bundles/org.openhab.binding.liquidcheck/src/main/java/org/openhab/binding/liquidcheck/internal/httpclient/LiquidCheckHttpClient.java

```diff
@@ -0,0 +1,106 @@
+/**
+ * Copyright (c) 2010-2023 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.liquidcheck.internal.httpclient;
+
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jetty.client.HttpClient;
+import org.eclipse.jetty.client.api.ContentResponse;
+import org.eclipse.jetty.client.api.Request;
+import org.eclipse.jetty.client.util.StringContentProvider;
+import org.eclipse.jetty.http.HttpHeader;
+import org.eclipse.jetty.http.HttpMethod;
+import org.openhab.binding.liquidcheck.internal.LiquidCheckConfiguration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link LiquidCheckHttpClient} sets up the jetty client for the connection to the device.
+ *
+ * @author Marcel Goerentz - Initial contribution
+ */
+@NonNullByDefault
+public class LiquidCheckHttpClient {
+
+    private final Logger logger = LoggerFactory.getLogger(LiquidCheckHttpClient.class);
+    private final HttpClient client;
+    private final LiquidCheckConfiguration config;
+
+    public boolean isClosed = false;
+
+    /**
+     * The Constructor of the LiquidCheckHttpClient class will set up a jetty client
+     * 
+     * @param config
+     */
+    public LiquidCheckHttpClient(LiquidCheckConfiguration config, HttpClient client) {
+        this.config = config;
+        this.client = client;
+        if (!client.isStarted()) {
+            try {
+                client.start();
+            } catch (Exception e) {
+                logger.warn("Could not start client. {}", e.getMessage());
+            }
+        }
```

Comment:
Hey, at least on my set up this is necessary.
If this part of the code is missing then there is no communication between openHAB and the device.

Reply:
Yeah, I have the 4.0 instance only for testing purpose, so no other bindings are installed and there is it working as describes.
I'm running a 3.4.3 where other bindings are involved, and there I need to start the the client by myself...
I'll deleted it for 4.0.0

Reply:
On your 3.4 system, with the binding running and having the problem, can you try in the console to run;
```shell
threads --monitors --locks
```

? I'm concerned if this is a bug in the binding or something else happening on your system, e.g. misbehavior other another binding.

You could also try that on your 4.0 instance.

Reply:
@jlaur Here you go:
4.0.0 output:
<details>
"CM Configuration Updater" Id=34 in WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@64dc5e97
    at java.base@17.0.4/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.4/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
    at java.base@17.0.4/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)
    at java.base@17.0.4/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3463)
    at java.base@17.0.4/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3434)
    at java.base@17.0.4/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623)
    at java.base@17.0.4/java.util.concurrent.LinkedBlockingDeque.takeFirst(LinkedBlockingDeque.java:485)
    at java.base@17.0.4/java.util.concurrent.LinkedBlockingDeque.take(LinkedBlockingDeque.java:673)
    at org.apache.felix.cm.impl.UpdateThread.run(UpdateThread.java:73)
    at java.base@17.0.4/java.lang.Thread.run(Thread.java:833)

"CM Event Dispatcher" Id=35 in WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@5eda6661
    at java.base@17.0.4/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.4/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
    at java.base@17.0.4/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)
    at java.base@17.0.4/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3463)
    at java.base@17.0.4/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3434)
    at java.base@17.0.4/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623)
    at java.base@17.0.4/java.util.concurrent.LinkedBlockingDeque.takeFirst(LinkedBlockingDeque.java:485)
    at java.base@17.0.4/java.util.concurrent.LinkedBlockingDeque.take(LinkedBlockingDeque.java:673)
    at org.apache.felix.cm.impl.UpdateThread.run(UpdateThread.java:73)
    at java.base@17.0.4/java.lang.Thread.run(Thread.java:833)

"Log4j2-TF-5-Scheduled-3" Id=50 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@3bd523a4
    at java.base@17.0.4/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.4/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.4/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at java.base@17.0.4/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1182)
    at java.base@17.0.4/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.4/java.lang.Thread.run(Thread.java:833)

"fileinstall-C:\Git\OPENHA~1.0-S/addons" Id=51 in TIMED_WAITING
    at java.base@17.0.4/java.lang.Thread.sleep(Native Method)
    at org.apache.felix.fileinstall.internal.DirectoryWatcher.run(DirectoryWatcher.java:291)

"encryption-1-thread-1" Id=76 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@36b1b3d1
    at java.base@17.0.4/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.4/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.4/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at java.base@17.0.4/java.util.concurrent.LinkedBlockingDeque.pollFirst(LinkedBlockingDeque.java:515)
    at java.base@17.0.4/java.util.concurrent.LinkedBlockingDeque.poll(LinkedBlockingDeque.java:677)
    at java.base@17.0.4/sun.nio.fs.AbstractWatchService.poll(AbstractWatchService.java:108)
    at org.apache.karaf.jaas.modules.properties.AutoEncryptionSupport.run(AutoEncryptionSupport.java:94)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.4/java.lang.Thread.run(Thread.java:833)

    Locked synchronizers: count = 1
      - java.util.concurrent.ThreadPoolExecutor$Worker@2b6b775b

"main" Id=1 in WAITING on lock=java.util.concurrent.atomic.AtomicReference@65f34d22
    at java.base@17.0.4/java.lang.Object.wait(Native Method)
    at org.eclipse.osgi.container.SystemModule.waitForStop(SystemModule.java:173)
    at org.eclipse.osgi.internal.framework.EquinoxBundle$SystemBundle.waitForStop(EquinoxBundle.java:312)
    at org.eclipse.osgi.launch.Equinox.waitForStop(Equinox.java:217)
    at app//org.apache.karaf.main.Main.awaitShutdown(Main.java:704)
    at app//org.apache.karaf.main.Main.main(Main.java:193)

"Active Thread: Equinox Container: bdb8cefc-74e6-4812-a8f2-3314f60d8155" Id=16 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@4917f363
    at java.base@17.0.4/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.4/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.4/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at java.base@17.0.4/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1182)
    at java.base@17.0.4/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.4/java.lang.Thread.run(Thread.java:833)

"Framework Event Dispatcher: Equinox Container: bdb8cefc-74e6-4812-a8f2-3314f60d8155" Id=18 in WAITING on lock=org.eclipse.osgi.framework.eventmgr.EventManager$EventThread@36f22d3e
    at java.base@17.0.4/java.lang.Object.wait(Native Method)
    at java.base@17.0.4/java.lang.Object.wait(Object.java:338)
    at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.getNextEvent(EventManager.java:400)
    at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:341)

"Bundle File Closer" Id=19 in WAITING on lock=org.eclipse.osgi.framework.eventmgr.EventManager$EventThread@5439752e
    at java.base@17.0.4/java.lang.Object.wait(Native Method)
    at java.base@17.0.4/java.lang.Object.wait(Object.java:338)
    at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.getNextEvent(EventManager.java:400)
    at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:341)

"Start Level: Equinox Container: bdb8cefc-74e6-4812-a8f2-3314f60d8155" Id=20 in RUNNABLE
    at java.base@17.0.4/java.lang.String.charAt(String.java:1515)
    at org.apache.felix.cm.impl.SimpleFilter.skipWhitespace(SimpleFilter.java:567)
    at org.apache.felix.cm.impl.SimpleFilter.parse(SimpleFilter.java:294)
    at org.apache.felix.cm.impl.ConfigurationManager.listConfigurations(ConfigurationManager.java:526)
    at org.apache.felix.cm.impl.ConfigurationAdminImpl.listConfigurations(ConfigurationAdminImpl.java:190)
    at org.apache.felix.scr.impl.manager.RegionConfigurationSupport.findConfigurations(RegionConfigurationSupport.java:737)
    at org.apache.felix.scr.impl.manager.RegionConfigurationSupport.findSingletonConfiguration(RegionConfigurationSupport.java:603)
    at org.apache.felix.scr.impl.manager.RegionConfigurationSupport.configureComponentHolder(RegionConfigurationSupport.java:219)
    at org.apache.felix.scr.impl.BundleComponentActivator.setRegionConfigurationSupport(BundleComponentActivator.java:751)
    at org.apache.felix.scr.impl.helper.ConfigAdminTracker$1.addingService(ConfigAdminTracker.java:69)
    at org.apache.felix.scr.impl.helper.ConfigAdminTracker$1.addingService(ConfigAdminTracker.java:41)
    at app//org.osgi.util.tracker.ServiceTracker$Tracked.customizerAdding(ServiceTracker.java:943)
    at app//org.osgi.util.tracker.ServiceTracker$Tracked.customizerAdding(ServiceTracker.java:871)
    at app//org.osgi.util.tracker.AbstractTracked.trackAdding(AbstractTracked.java:256)
    at app//org.osgi.util.tracker.AbstractTracked.trackInitial(AbstractTracked.java:183)
    at app//org.osgi.util.tracker.ServiceTracker.open(ServiceTracker.java:321)
    at app//org.osgi.util.tracker.ServiceTracker.open(ServiceTracker.java:264)
    at org.apache.felix.scr.impl.helper.ConfigAdminTracker.<init>(ConfigAdminTracker.java:86)
    at org.apache.felix.scr.impl.BundleComponentActivator.<init>(BundleComponentActivator.java:224)
    at org.apache.felix.scr.impl.Activator.loadComponents(Activator.java:592)
    at org.apache.felix.scr.impl.Activator.access$200(Activator.java:74)
    at org.apache.felix.scr.impl.Activator$ScrExtension.start(Activator.java:460)
    at org.apache.felix.scr.impl.AbstractExtender.createExtension(AbstractExtender.java:196)
    at org.apache.felix.scr.impl.AbstractExtender.modifiedBundle(AbstractExtender.java:169)
    at org.apache.felix.scr.impl.AbstractExtender.modifiedBundle(AbstractExtender.java:49)
    at app//org.osgi.util.tracker.BundleTracker$Tracked.customizerModified(BundleTracker.java:488)
    at app//org.osgi.util.tracker.BundleTracker$Tracked.customizerModified(BundleTracker.java:420)
    at app//org.osgi.util.tracker.AbstractTracked.track(AbstractTracked.java:232)
    at app//org.osgi.util.tracker.BundleTracker$Tracked.bundleChanged(BundleTracker.java:450)
    at org.eclipse.osgi.internal.framework.BundleContextImpl.dispatchEvent(BundleContextImpl.java:949)
    at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:234)
    at org.eclipse.osgi.framework.eventmgr.ListenerQueue.dispatchEventSynchronous(ListenerQueue.java:151)
    at org.eclipse.osgi.internal.framework.EquinoxEventPublisher.publishBundleEventPrivileged(EquinoxEventPublisher.java:229)
    at org.eclipse.osgi.internal.framework.EquinoxEventPublisher.publishBundleEvent(EquinoxEventPublisher.java:138)
    at org.eclipse.osgi.internal.framework.EquinoxEventPublisher.publishBundleEvent(EquinoxEventPublisher.java:130)
    at org.eclipse.osgi.internal.framework.EquinoxContainerAdaptor.publishModuleEvent(EquinoxContainerAdaptor.java:217)
    at org.eclipse.osgi.container.Module.publishEvent(Module.java:499)
    at org.eclipse.osgi.container.Module.start(Module.java:486)
    at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel$2.run(ModuleContainer.java:1847)
    at org.eclipse.osgi.internal.framework.EquinoxContainerAdaptor$1$1.execute(EquinoxContainerAdaptor.java:136)
    at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1840)
    at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1783)
    at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.doContainerStartLevel(ModuleContainer.java:1745)
      - locked java.lang.Object@35c55155
    at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1667)
    at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1)
    at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:234)
    at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:345)

    Locked synchronizers: count = 1
      - java.util.concurrent.locks.ReentrantLock$NonfairSync@433f2065

    Locked monitors: count = 1
      - java.lang.Object@35c55155 locked at
          42 org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.doContainerStartLevel(ModuleContainer.java:1745)

"Karaf Lock Monitor Thread" Id=21 in TIMED_WAITING
    at java.base@17.0.4/java.lang.Thread.sleep(Native Method)
    at app//org.apache.karaf.main.Main.doMonitor(Main.java:442)
    at app//org.apache.karaf.main.Main.access$100(Main.java:78)
    at app//org.apache.karaf.main.Main$3.run(Main.java:409)

"Karaf Shutdown Socket Thread" Id=22 in RUNNABLE (running in native)
    at java.base@17.0.4/sun.nio.ch.Net.accept(Native Method)
    at java.base@17.0.4/sun.nio.ch.NioSocketImpl.accept(NioSocketImpl.java:755)
    at java.base@17.0.4/java.net.ServerSocket.implAccept(ServerSocket.java:686)
    at java.base@17.0.4/java.net.ServerSocket.platformImplAccept(ServerSocket.java:652)
    at java.base@17.0.4/java.net.ServerSocket.implAccept(ServerSocket.java:628)
    at java.base@17.0.4/java.net.ServerSocket.implAccept(ServerSocket.java:585)
    at java.base@17.0.4/java.net.ServerSocket.accept(ServerSocket.java:538)
    at app//org.apache.karaf.main.ShutdownSocketThread.run(ShutdownSocketThread.java:66)

    Locked synchronizers: count = 1
      - java.util.concurrent.locks.ReentrantLock$NonfairSync@64837ee1

"fileinstall-C:\Git\openhab-4.0.0-SNAPSHOT\userdata/etc" Id=41 in TIMED_WAITING on lock=org.apache.felix.fileinstall.internal.DirectoryWatcher@21516f2d
    at java.base@17.0.4/java.lang.Object.wait(Native Method)
    at org.apache.felix.fileinstall.internal.DirectoryWatcher.run(DirectoryWatcher.java:320)

"EventAdminAsyncThread #11" Id=44 in WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@3d7bea7a
    at java.base@17.0.4/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.4/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
    at java.base@17.0.4/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)
    at java.base@17.0.4/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3463)
    at java.base@17.0.4/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3434)
    at java.base@17.0.4/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623)
    at java.base@17.0.4/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.4/java.lang.Thread.run(Thread.java:833)

"EventAdminAsyncThread #12" Id=46 in WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@3d7bea7a
    at java.base@17.0.4/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.4/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
    at java.base@17.0.4/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)
    at java.base@17.0.4/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3463)
    at java.base@17.0.4/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3434)
    at java.base@17.0.4/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623)
    at java.base@17.0.4/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.4/java.lang.Thread.run(Thread.java:833)

"encryption-3-thread-1" Id=80 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@1cb39a2d
    at java.base@17.0.4/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.4/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.4/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at java.base@17.0.4/java.util.concurrent.LinkedBlockingDeque.pollFirst(LinkedBlockingDeque.java:515)
    at java.base@17.0.4/java.util.concurrent.LinkedBlockingDeque.poll(LinkedBlockingDeque.java:677)
    at java.base@17.0.4/sun.nio.fs.AbstractWatchService.poll(AbstractWatchService.java:108)
    at org.apache.karaf.jaas.modules.properties.AutoEncryptionSupport.run(AutoEncryptionSupport.java:94)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.4/java.lang.Thread.run(Thread.java:833)

    Locked synchronizers: count = 1
      - java.util.concurrent.ThreadPoolExecutor$Worker@16aae7b6

"JNA Cleaner" Id=86 in WAITING on lock=java.lang.ref.ReferenceQueue$Lock@7e798bfb
    at java.base@17.0.4/java.lang.Object.wait(Native Method)
    at java.base@17.0.4/java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:155)
    at java.base@17.0.4/java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:176)
    at com.sun.jna.internal.Cleaner$1.run(Cleaner.java:58)

"WindowsStreamPump" Id=88 in RUNNABLE (running in native)
    at com.sun.jna.Native.invokeInt(Native Method)
    at com.sun.jna.Function.invoke(Function.java:426)
    at com.sun.jna.Function.invoke(Function.java:361)
    at com.sun.jna.Library$Handler.invoke(Library.java:265)
    at org.jline.terminal.impl.jna.win.$Proxy52.WaitForSingleObject(Unknown Source)
    at org.jline.terminal.impl.jna.win.JnaWinSysTerminal.readConsoleInput(JnaWinSysTerminal.java:207)
    at org.jline.terminal.impl.jna.win.JnaWinSysTerminal.processConsoleInput(JnaWinSysTerminal.java:130)
    at org.jline.terminal.impl.AbstractWindowsTerminal.pump(AbstractWindowsTerminal.java:460)
    at org.jline.terminal.impl.AbstractWindowsTerminal$$Lambda$272/0x00000007c04e2f30.run(Unknown Source)
    at java.base@17.0.4/java.lang.Thread.run(Thread.java:833)

"Karaf local console user openhab" Id=89 in WAITING on lock=org.apache.felix.gogo.runtime.CommandSessionImpl$JobImpl@6ca35f93
    at java.base@17.0.4/java.lang.Object.wait(Native Method)
    at java.base@17.0.4/java.lang.Object.wait(Object.java:338)
    at org.apache.felix.gogo.runtime.CommandSessionImpl$JobImpl.start(CommandSessionImpl.java:802)
    at org.apache.felix.gogo.runtime.Closure.execute(Closure.java:300)
    at org.apache.felix.gogo.runtime.Closure.execute(Closure.java:169)
    at org.apache.felix.gogo.runtime.Closure.execute(Closure.java:149)
    at org.apache.felix.gogo.runtime.CommandSessionImpl.execute(CommandSessionImpl.java:179)
    at org.apache.karaf.shell.impl.console.ConsoleSessionImpl.doExecute(ConsoleSessionImpl.java:475)
    at org.apache.karaf.shell.impl.console.ConsoleSessionImpl.run(ConsoleSessionImpl.java:410)
    at java.base@17.0.4/java.lang.Thread.run(Thread.java:833)

"job controller 1" Id=96 in WAITING on lock=java.util.concurrent.FutureTask@573756ed
    at java.base@17.0.4/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.4/java.util.concurrent.locks.LockSupport.park(LockSupport.java:211)
    at java.base@17.0.4/java.util.concurrent.FutureTask.awaitDone(FutureTask.java:447)
    at java.base@17.0.4/java.util.concurrent.FutureTask.get(FutureTask.java:190)
    at java.base@17.0.4/java.util.concurrent.AbstractExecutorService.invokeAll(AbstractExecutorService.java:252)
    at org.apache.felix.gogo.runtime.CommandSessionImpl$JobImpl.run(CommandSessionImpl.java:827)
    at java.base@17.0.4/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
    at java.base@17.0.4/java.util.concurrent.FutureTask.run(FutureTask.java:264)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.4/java.lang.Thread.run(Thread.java:833)

    Locked synchronizers: count = 1
      - java.util.concurrent.ThreadPoolExecutor$Worker@4961d57d

"session-1-thread-2" Id=98 in TIMED_WAITING on lock=java.util.concurrent.SynchronousQueue$TransferStack@57d5ccd4
    at java.base@17.0.4/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.4/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.4/java.util.concurrent.SynchronousQueue$TransferStack.transfer(SynchronousQueue.java:401)
    at java.base@17.0.4/java.util.concurrent.SynchronousQueue.poll(SynchronousQueue.java:903)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1061)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.4/java.lang.Thread.run(Thread.java:833)

"pipe-threads --monitors --locks" Id=107 in RUNNABLE
    at java.management@17.0.4/sun.management.ThreadImpl.dumpThreads0(Native Method)
    at java.management@17.0.4/sun.management.ThreadImpl.dumpAllThreads(ThreadImpl.java:521)
    at java.management@17.0.4/sun.management.ThreadImpl.dumpAllThreads(ThreadImpl.java:509)
    at org.apache.karaf.shell.commands.impl.ThreadsAction.execute(ThreadsAction.java:78)
    at org.apache.karaf.shell.impl.action.command.ActionCommand.execute(ActionCommand.java:84)
    at org.apache.karaf.shell.impl.console.osgi.secured.SecuredCommand.execute(SecuredCommand.java:68)
    at org.apache.karaf.shell.impl.console.osgi.secured.SecuredCommand.execute(SecuredCommand.java:86)
    at org.apache.felix.gogo.runtime.Closure.executeCmd(Closure.java:599)
    at org.apache.felix.gogo.runtime.Closure.executeStatement(Closure.java:526)
    at org.apache.felix.gogo.runtime.Closure.execute(Closure.java:415)
    at org.apache.felix.gogo.runtime.Pipe.doCall(Pipe.java:416)
    at org.apache.felix.gogo.runtime.Pipe.call(Pipe.java:229)
    at org.apache.felix.gogo.runtime.Pipe.call(Pipe.java:59)
    at java.base@17.0.4/java.util.concurrent.FutureTask.run(FutureTask.java:264)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.4/java.lang.Thread.run(Thread.java:833)

    Locked synchronizers: count = 1
      - java.util.concurrent.ThreadPoolExecutor$Worker@19a05b81

"session-1-thread-4" Id=109 in TIMED_WAITING on lock=java.util.concurrent.SynchronousQueue$TransferStack@57d5ccd4
    at java.base@17.0.4/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.4/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.4/java.util.concurrent.SynchronousQueue$TransferStack.transfer(SynchronousQueue.java:401)
    at java.base@17.0.4/java.util.concurrent.SynchronousQueue.poll(SynchronousQueue.java:903)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1061)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.4/java.lang.Thread.run(Thread.java:833)

"SCR Component Actor" Id=110 in WAITING on lock=java.util.LinkedList@65c1a386
    at java.base@17.0.4/java.lang.Object.wait(Native Method)
    at java.base@17.0.4/java.lang.Object.wait(Object.java:338)
    at org.apache.felix.scr.impl.ComponentActorThread.run(ComponentActorThread.java:83)
    at java.base@17.0.4/java.lang.Thread.run(Thread.java:833)

"sshd-SshServer[57fdfdc7](port=8101)-timer-thread-1" Id=195 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@7e059bd4
    at java.base@17.0.4/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.4/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.4/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at java.base@17.0.4/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1182)
    at java.base@17.0.4/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.4/java.lang.Thread.run(Thread.java:833)

"paxweb-config-1-thread-1" Id=198 in WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@7cf5bac
    at java.base@17.0.4/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.4/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
    at java.base@17.0.4/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)
    at java.base@17.0.4/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3463)
    at java.base@17.0.4/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3434)
    at java.base@17.0.4/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623)
    at java.base@17.0.4/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1170)
    at java.base@17.0.4/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.4/java.lang.Thread.run(Thread.java:833)

"qtp1349827412-283" Id=283 in RUNNABLE (running in native)
    at java.base@17.0.4/sun.nio.ch.WEPoll.wait(Native Method)
    at java.base@17.0.4/sun.nio.ch.WEPollSelectorImpl.doSelect(WEPollSelectorImpl.java:111)
    at java.base@17.0.4/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - locked sun.nio.ch.Util$2@6ffa0540
      - locked sun.nio.ch.WEPollSelectorImpl@b60cd25
    at java.base@17.0.4/sun.nio.ch.SelectorImpl.select(SelectorImpl.java:146)
    at org.eclipse.jetty.io.ManagedSelector.nioSelect(ManagedSelector.java:183)
    at org.eclipse.jetty.io.ManagedSelector.select(ManagedSelector.java:190)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.select(ManagedSelector.java:606)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.produce(ManagedSelector.java:543)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produceTask(EatWhatYouKill.java:362)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:186)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:173)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produce(EatWhatYouKill.java:137)
    at org.eclipse.jetty.io.ManagedSelector$$Lambda$737/0x00000007c0651e88.run(Unknown Source)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.4/java.lang.Thread.run(Thread.java:833)

    Locked monitors: count = 2
      - sun.nio.ch.Util$2@6ffa0540 locked at
          2 java.base@17.0.4/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - sun.nio.ch.WEPollSelectorImpl@b60cd25 locked at
          2 java.base@17.0.4/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)

"qtp1349827412-284" Id=284 in RUNNABLE (running in native)
    at java.base@17.0.4/sun.nio.ch.WEPoll.wait(Native Method)
    at java.base@17.0.4/sun.nio.ch.WEPollSelectorImpl.doSelect(WEPollSelectorImpl.java:111)
    at java.base@17.0.4/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - locked sun.nio.ch.Util$2@3dfb2ca2
      - locked sun.nio.ch.WEPollSelectorImpl@3b587b23
    at java.base@17.0.4/sun.nio.ch.SelectorImpl.select(SelectorImpl.java:146)
    at org.eclipse.jetty.io.ManagedSelector.nioSelect(ManagedSelector.java:183)
    at org.eclipse.jetty.io.ManagedSelector.select(ManagedSelector.java:190)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.select(ManagedSelector.java:606)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.produce(ManagedSelector.java:543)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produceTask(EatWhatYouKill.java:362)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:186)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:173)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produce(EatWhatYouKill.java:137)
    at org.eclipse.jetty.io.ManagedSelector$$Lambda$737/0x00000007c0651e88.run(Unknown Source)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.4/java.lang.Thread.run(Thread.java:833)

    Locked monitors: count = 2
      - sun.nio.ch.Util$2@3dfb2ca2 locked at
          2 java.base@17.0.4/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - sun.nio.ch.WEPollSelectorImpl@3b587b23 locked at
          2 java.base@17.0.4/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)

"Session-HouseKeeper-2fcf5692-1" Id=335 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@6ff2cbf2
    at java.base@17.0.4/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.4/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.4/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at java.base@17.0.4/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1182)
    at java.base@17.0.4/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.4/java.lang.Thread.run(Thread.java:833)

"EMF Reference Cleaner" Id=336 in WAITING on lock=java.lang.ref.ReferenceQueue$Lock@121dff9b
    at java.base@17.0.4/java.lang.Object.wait(Native Method)
    at java.base@17.0.4/java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:155)
    at java.base@17.0.4/java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:176)
    at org.eclipse.emf.common.util.CommonUtil$1ReferenceClearingQueuePollingThread.run(CommonUtil.java:70)

"ForkJoinPool.commonPool-worker-1" Id=361 in TIMED_WAITING on lock=java.util.concurrent.ForkJoinPool@49089fad
    at java.base@17.0.4/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.4/java.util.concurrent.locks.LockSupport.parkUntil(LockSupport.java:410)
    at java.base@17.0.4/java.util.concurrent.ForkJoinPool.awaitWork(ForkJoinPool.java:1726)
    at java.base@17.0.4/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1623)
    at java.base@17.0.4/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)

"OH-OSGiEventManager" Id=362 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@30dc6025
    at java.base@17.0.4/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.4/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.4/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at java.base@17.0.4/java.util.concurrent.LinkedBlockingQueue.poll(LinkedBlockingQueue.java:460)
    at org.openhab.core.internal.events.ThreadedEventHandler.lambda$0(ThreadedEventHandler.java:60)
    at org.openhab.core.internal.events.ThreadedEventHandler$$Lambda$728/0x00000007c0704b10.run(Unknown Source)
    at java.base@17.0.4/java.lang.Thread.run(Thread.java:833)

"qtp1349827412-363" Id=363 in RUNNABLE (running in native)
    at java.base@17.0.4/sun.nio.ch.WEPoll.wait(Native Method)
    at java.base@17.0.4/sun.nio.ch.WEPollSelectorImpl.doSelect(WEPollSelectorImpl.java:111)
    at java.base@17.0.4/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - locked sun.nio.ch.Util$2@4dc0a082
      - locked sun.nio.ch.WEPollSelectorImpl@a306c83
    at java.base@17.0.4/sun.nio.ch.SelectorImpl.select(SelectorImpl.java:146)
    at org.eclipse.jetty.io.ManagedSelector.nioSelect(ManagedSelector.java:183)
    at org.eclipse.jetty.io.ManagedSelector.select(ManagedSelector.java:190)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.select(ManagedSelector.java:606)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.produce(ManagedSelector.java:543)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produceTask(EatWhatYouKill.java:362)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:186)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:173)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produce(EatWhatYouKill.java:137)
    at org.eclipse.jetty.io.ManagedSelector$$Lambda$737/0x00000007c0651e88.run(Unknown Source)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.4/java.lang.Thread.run(Thread.java:833)

    Locked monitors: count = 2
      - sun.nio.ch.Util$2@4dc0a082 locked at
          2 java.base@17.0.4/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - sun.nio.ch.WEPollSelectorImpl@a306c83 locked at
          2 java.base@17.0.4/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)

"qtp1349827412-364-acceptor-0@2a9fae00-0.0.0.0:8443@288cdd5c{SSL, (ssl, http/1.1)}{0.0.0.0:8443}" Id=364 in RUNNABLE (running in native)
    at java.base@17.0.4/sun.nio.ch.Net.accept(Native Method)
    at java.base@17.0.4/sun.nio.ch.ServerSocketChannelImpl.implAccept(ServerSocketChannelImpl.java:425)
    at java.base@17.0.4/sun.nio.ch.ServerSocketChannelImpl.accept(ServerSocketChannelImpl.java:391)
    at org.eclipse.jetty.server.ServerConnector.accept(ServerConnector.java:388)
    at org.eclipse.jetty.server.AbstractConnector$Acceptor.run(AbstractConnector.java:704)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.4/java.lang.Thread.run(Thread.java:833)

    Locked synchronizers: count = 1
      - java.util.concurrent.locks.ReentrantLock$NonfairSync@71ca4f67

"qtp1349827412-365" Id=365 in RUNNABLE (running in native)
    at java.base@17.0.4/sun.nio.ch.WEPoll.wait(Native Method)
    at java.base@17.0.4/sun.nio.ch.WEPollSelectorImpl.doSelect(WEPollSelectorImpl.java:111)
    at java.base@17.0.4/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - locked sun.nio.ch.Util$2@7c325304
      - locked sun.nio.ch.WEPollSelectorImpl@62c09e3f
    at java.base@17.0.4/sun.nio.ch.SelectorImpl.select(SelectorImpl.java:146)
    at org.eclipse.jetty.io.ManagedSelector.nioSelect(ManagedSelector.java:183)
    at org.eclipse.jetty.io.ManagedSelector.select(ManagedSelector.java:190)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.select(ManagedSelector.java:606)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.produce(ManagedSelector.java:543)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produceTask(EatWhatYouKill.java:362)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:186)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:173)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produce(EatWhatYouKill.java:137)
    at org.eclipse.jetty.io.ManagedSelector$$Lambda$737/0x00000007c0651e88.run(Unknown Source)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.4/java.lang.Thread.run(Thread.java:833)

    Locked monitors: count = 2
      - sun.nio.ch.Util$2@7c325304 locked at
          2 java.base@17.0.4/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - sun.nio.ch.WEPollSelectorImpl@62c09e3f locked at
          2 java.base@17.0.4/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)

"qtp1349827412-366" Id=366 in RUNNABLE (running in native)
    at java.base@17.0.4/sun.nio.ch.WEPoll.wait(Native Method)
    at java.base@17.0.4/sun.nio.ch.WEPollSelectorImpl.doSelect(WEPollSelectorImpl.java:111)
    at java.base@17.0.4/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - locked sun.nio.ch.Util$2@2f3cd9ef
      - locked sun.nio.ch.WEPollSelectorImpl@55f6501b
    at java.base@17.0.4/sun.nio.ch.SelectorImpl.select(SelectorImpl.java:146)
    at org.eclipse.jetty.io.ManagedSelector.nioSelect(ManagedSelector.java:183)
    at org.eclipse.jetty.io.ManagedSelector.select(ManagedSelector.java:190)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.select(ManagedSelector.java:606)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.produce(ManagedSelector.java:543)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produceTask(EatWhatYouKill.java:362)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:186)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:173)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produce(EatWhatYouKill.java:137)
    at org.eclipse.jetty.io.ManagedSelector$$Lambda$737/0x00000007c0651e88.run(Unknown Source)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.4/java.lang.Thread.run(Thread.java:833)

    Locked monitors: count = 2
      - sun.nio.ch.Util$2@2f3cd9ef locked at
          2 java.base@17.0.4/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - sun.nio.ch.WEPollSelectorImpl@55f6501b locked at
          2 java.base@17.0.4/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)

"qtp1349827412-367" Id=367 in RUNNABLE (running in native)
    at java.base@17.0.4/sun.nio.ch.WEPoll.wait(Native Method)
    at java.base@17.0.4/sun.nio.ch.WEPollSelectorImpl.doSelect(WEPollSelectorImpl.java:111)
    at java.base@17.0.4/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - locked sun.nio.ch.Util$2@6730a83a
      - locked sun.nio.ch.WEPollSelectorImpl@6ed5dbcb
    at java.base@17.0.4/sun.nio.ch.SelectorImpl.select(SelectorImpl.java:146)
    at org.eclipse.jetty.io.ManagedSelector.nioSelect(ManagedSelector.java:183)
    at org.eclipse.jetty.io.ManagedSelector.select(ManagedSelector.java:190)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.select(ManagedSelector.java:606)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.produce(ManagedSelector.java:543)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produceTask(EatWhatYouKill.java:362)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:186)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:173)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produce(EatWhatYouKill.java:137)
    at org.eclipse.jetty.io.ManagedSelector$$Lambda$737/0x00000007c0651e88.run(Unknown Source)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.4/java.lang.Thread.run(Thread.java:833)

    Locked monitors: count = 2
      - sun.nio.ch.Util$2@6730a83a locked at
          2 java.base@17.0.4/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - sun.nio.ch.WEPollSelectorImpl@6ed5dbcb locked at
          2 java.base@17.0.4/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)

"qtp1349827412-368-acceptor-0@77454e9b-default@2218928e{HTTP/1.1, (http/1.1, h2c)}{0.0.0.0:8080}" Id=368 in RUNNABLE (running in native)
    at java.base@17.0.4/sun.nio.ch.Net.accept(Native Method)
    at java.base@17.0.4/sun.nio.ch.ServerSocketChannelImpl.implAccept(ServerSocketChannelImpl.java:425)
    at java.base@17.0.4/sun.nio.ch.ServerSocketChannelImpl.accept(ServerSocketChannelImpl.java:391)
    at org.eclipse.jetty.server.ServerConnector.accept(ServerConnector.java:388)
    at org.eclipse.jetty.server.AbstractConnector$Acceptor.run(AbstractConnector.java:704)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.4/java.lang.Thread.run(Thread.java:833)

    Locked synchronizers: count = 1
      - java.util.concurrent.locks.ReentrantLock$NonfairSync@442272cc

"HttpService->Whiteboard" Id=370 in WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@2ae6a33f
    at java.base@17.0.4/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.4/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
    at java.base@17.0.4/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)
    at java.base@17.0.4/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3463)
    at java.base@17.0.4/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3434)
    at java.base@17.0.4/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623)
    at java.base@17.0.4/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.4/java.lang.Thread.run(Thread.java:833)

"OH-common-1" Id=371 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@63dcf42d
    at java.base@17.0.4/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.4/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.4/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at java.base@17.0.4/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.poll(ScheduledThreadPoolExecutor.java:1218)
    at java.base@17.0.4/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.poll(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1061)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.4/java.lang.Thread.run(Thread.java:833)

"OH-common-2" Id=373 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@63dcf42d
    at java.base@17.0.4/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.4/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.4/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at java.base@17.0.4/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.poll(ScheduledThreadPoolExecutor.java:1218)
    at java.base@17.0.4/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.poll(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1061)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.4/java.lang.Thread.run(Thread.java:833)

"OH-startlevel-1" Id=374 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@1cfe114
    at java.base@17.0.4/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.4/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.4/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at java.base@17.0.4/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1182)
    at java.base@17.0.4/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.4/java.lang.Thread.run(Thread.java:833)

"OH-common-3" Id=377 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@63dcf42d
    at java.base@17.0.4/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.4/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.4/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at java.base@17.0.4/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.poll(ScheduledThreadPoolExecutor.java:1218)
    at java.base@17.0.4/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.poll(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1061)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.4/java.lang.Thread.run(Thread.java:833)

"configWatcher" Id=378 in WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@562c57f3
    at java.base@17.0.4/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.4/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
    at java.base@17.0.4/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)
    at java.base@17.0.4/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3463)
    at java.base@17.0.4/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3434)
    at java.base@17.0.4/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623)
    at java.base@17.0.4/java.util.concurrent.LinkedBlockingDeque.takeFirst(LinkedBlockingDeque.java:485)
    at java.base@17.0.4/java.util.concurrent.LinkedBlockingDeque.take(LinkedBlockingDeque.java:673)
    at java.base@17.0.4/sun.nio.fs.AbstractWatchService.take(AbstractWatchService.java:118)
    at io.methvin.watcher.DirectoryWatcher.runEventLoop(DirectoryWatcher.java:281)
    at io.methvin.watcher.DirectoryWatcher.lambda$watchAsync$1(DirectoryWatcher.java:232)
    at io.methvin.watcher.DirectoryWatcher$$Lambda$934/0x00000007c07cc000.get(Unknown Source)
    at java.base@17.0.4/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.4/java.lang.Thread.run(Thread.java:833)

    Locked synchronizers: count = 1
      - java.util.concurrent.ThreadPoolExecutor$Worker@1fe738f6

"OH-common-4" Id=379 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@63dcf42d
    at java.base@17.0.4/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.4/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.4/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at java.base@17.0.4/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.poll(ScheduledThreadPoolExecutor.java:1223)
    at java.base@17.0.4/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.poll(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1061)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.4/java.lang.Thread.run(Thread.java:833)

"OH-common-5" Id=380 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@63dcf42d
    at java.base@17.0.4/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.4/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.4/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at java.base@17.0.4/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.poll(ScheduledThreadPoolExecutor.java:1223)
    at java.base@17.0.4/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.poll(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1061)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.4/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.4/java.lang.Thread.run(Thread.java:833)

"Common-Cleaner" Id=13 in TIMED_WAITING on lock=java.lang.ref.ReferenceQueue$Lock@5b475b5f
    at java.base@17.0.4/java.lang.Object.wait(Native Method)
    at java.base@17.0.4/java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:155)
    at java.base@17.0.4/jdk.internal.ref.CleanerImpl.run(CleanerImpl.java:140)
    at java.base@17.0.4/java.lang.Thread.run(Thread.java:833)
    at java.base@17.0.4/jdk.internal.misc.InnocuousThread.run(InnocuousThread.java:162)
</details>

Reply:
3.4.0 output:
<details>
"CM Configuration Updater" Id=33 in WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@7a491483
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3463)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3434)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623)
    at java.base@17.0.7/java.util.concurrent.LinkedBlockingDeque.takeFirst(LinkedBlockingDeque.java:485)
    at java.base@17.0.7/java.util.concurrent.LinkedBlockingDeque.take(LinkedBlockingDeque.java:673)
    at org.apache.felix.cm.impl.UpdateThread.run(UpdateThread.java:73)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"CM Event Dispatcher" Id=34 in WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@49572edb
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3463)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3434)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623)
    at java.base@17.0.7/java.util.concurrent.LinkedBlockingDeque.takeFirst(LinkedBlockingDeque.java:485)
    at java.base@17.0.7/java.util.concurrent.LinkedBlockingDeque.take(LinkedBlockingDeque.java:673)
    at org.apache.felix.cm.impl.UpdateThread.run(UpdateThread.java:73)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"Log4j2-TF-5-Scheduled-3" Id=122 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@3430c7e2
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1182)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"fileinstall-/usr/share/openhab/addons" Id=127 in TIMED_WAITING on lock=org.apache.felix.fileinstall.internal.DirectoryWatcher@50fd3761
    at java.base@17.0.7/java.lang.Object.wait(Native Method)
    at org.apache.felix.fileinstall.internal.DirectoryWatcher.run(DirectoryWatcher.java:320)

"encryption-1-thread-1" Id=129 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@517a14ca
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at java.base@17.0.7/java.util.concurrent.LinkedBlockingDeque.pollFirst(LinkedBlockingDeque.java:515)
    at java.base@17.0.7/java.util.concurrent.LinkedBlockingDeque.poll(LinkedBlockingDeque.java:677)
    at java.base@17.0.7/sun.nio.fs.AbstractWatchService.poll(AbstractWatchService.java:108)
    at org.apache.karaf.jaas.modules.properties.AutoEncryptionSupport.run(AutoEncryptionSupport.java:94)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

    Locked synchronizers: count = 1
      - java.util.concurrent.ThreadPoolExecutor$Worker@4b0b13cf

"main" Id=1 in WAITING on lock=java.util.concurrent.atomic.AtomicReference@58f6c0fa
    at java.base@17.0.7/java.lang.Object.wait(Native Method)
    at org.eclipse.osgi.container.SystemModule.waitForStop(SystemModule.java:173)
    at org.eclipse.osgi.internal.framework.EquinoxBundle$SystemBundle.waitForStop(EquinoxBundle.java:312)
    at org.eclipse.osgi.launch.Equinox.waitForStop(Equinox.java:217)
    at app//org.apache.karaf.main.Main.awaitShutdown(Main.java:704)
    at app//org.apache.karaf.main.Main.main(Main.java:193)

"Active Thread: Equinox Container: 750bd21e-d2cb-4333-b20c-1cb88cf541af" Id=14 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@5b75cb35
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1182)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"Framework Event Dispatcher: Equinox Container: 750bd21e-d2cb-4333-b20c-1cb88cf541af" Id=16 in WAITING on lock=org.eclipse.osgi.framework.eventmgr.EventManager$EventThread@48e64e92
    at java.base@17.0.7/java.lang.Object.wait(Native Method)
    at java.base@17.0.7/java.lang.Object.wait(Object.java:338)
    at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.getNextEvent(EventManager.java:400)
    at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:341)

"Bundle File Closer" Id=17 in WAITING on lock=org.eclipse.osgi.framework.eventmgr.EventManager$EventThread@79a94099
    at java.base@17.0.7/java.lang.Object.wait(Native Method)
    at java.base@17.0.7/java.lang.Object.wait(Object.java:338)
    at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.getNextEvent(EventManager.java:400)
    at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:341)

"Start Level: Equinox Container: 750bd21e-d2cb-4333-b20c-1cb88cf541af" Id=18 in WAITING on lock=org.eclipse.osgi.framework.eventmgr.EventManager$EventThread@620c3a4d
    at java.base@17.0.7/java.lang.Object.wait(Native Method)
    at java.base@17.0.7/java.lang.Object.wait(Object.java:338)
    at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.getNextEvent(EventManager.java:400)
    at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:341)

"Karaf Lock Monitor Thread" Id=19 in TIMED_WAITING
    at java.base@17.0.7/java.lang.Thread.sleep(Native Method)
    at app//org.apache.karaf.main.Main.doMonitor(Main.java:442)
    at app//org.apache.karaf.main.Main.access$100(Main.java:78)
    at app//org.apache.karaf.main.Main$3.run(Main.java:409)

"Karaf Shutdown Socket Thread" Id=20 in RUNNABLE (running in native)
    at java.base@17.0.7/sun.nio.ch.Net.accept(Native Method)
    at java.base@17.0.7/sun.nio.ch.NioSocketImpl.accept(NioSocketImpl.java:755)
    at java.base@17.0.7/java.net.ServerSocket.implAccept(ServerSocket.java:686)
    at java.base@17.0.7/java.net.ServerSocket.platformImplAccept(ServerSocket.java:652)
    at java.base@17.0.7/java.net.ServerSocket.implAccept(ServerSocket.java:628)
    at java.base@17.0.7/java.net.ServerSocket.implAccept(ServerSocket.java:585)
    at java.base@17.0.7/java.net.ServerSocket.accept(ServerSocket.java:538)
    at app//org.apache.karaf.main.ShutdownSocketThread.run(ShutdownSocketThread.java:66)

    Locked synchronizers: count = 1
      - java.util.concurrent.locks.ReentrantLock$NonfairSync@75781932

"EventAdminAsyncThread #11" Id=36 in WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@7c531f4a
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3463)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3434)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623)
    at java.base@17.0.7/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EventAdminAsyncThread #12" Id=37 in WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@7c531f4a
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3463)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3434)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623)
    at java.base@17.0.7/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"fileinstall-/var/lib/openhab/etc" Id=43 in TIMED_WAITING on lock=org.apache.felix.fileinstall.internal.DirectoryWatcher@76bee70b
    at java.base@17.0.7/java.lang.Object.wait(Native Method)
    at org.apache.felix.fileinstall.internal.DirectoryWatcher.run(DirectoryWatcher.java:320)

"SCR Component Actor" Id=94 in WAITING on lock=java.util.LinkedList@44e11853
    at java.base@17.0.7/java.lang.Object.wait(Native Method)
    at java.base@17.0.7/java.lang.Object.wait(Object.java:338)
    at org.apache.felix.scr.impl.ComponentActorThread.run(ComponentActorThread.java:83)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"sshd-SshServer[370e5491](port=8101)-timer-thread-1" Id=194 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@1d15ad36
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1182)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EMF Reference Cleaner" Id=248 in WAITING on lock=java.lang.ref.ReferenceQueue$Lock@2f2866a1
    at java.base@17.0.7/java.lang.Object.wait(Native Method)
    at java.base@17.0.7/java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:155)
    at java.base@17.0.7/java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:176)
    at org.eclipse.emf.common.util.CommonUtil$1ReferenceClearingQueuePollingThread.run(CommonUtil.java:70)

"OH-OSGiEventManager" Id=274 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@16570a1a
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at java.base@17.0.7/java.util.concurrent.LinkedBlockingQueue.poll(LinkedBlockingQueue.java:460)
    at org.openhab.core.internal.events.ThreadedEventHandler.lambda$0(ThreadedEventHandler.java:60)
    at org.openhab.core.internal.events.ThreadedEventHandler$$Lambda$597/0x00000007c06ab0d8.run(Unknown Source)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"OH-startlevel-1" Id=276 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@178a805e
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1182)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"openHAB Dir Watcher" Id=298 in WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@27c6f384
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3463)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3434)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623)
    at java.base@17.0.7/java.util.concurrent.LinkedBlockingDeque.takeFirst(LinkedBlockingDeque.java:485)
    at java.base@17.0.7/java.util.concurrent.LinkedBlockingDeque.take(LinkedBlockingDeque.java:673)
    at java.base@17.0.7/sun.nio.fs.AbstractWatchService.take(AbstractWatchService.java:118)
    at org.openhab.core.service.WatchQueueReader.run(WatchQueueReader.java:245)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"ServletEventDispatcher: 1" Id=324 in WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@4b3794b3
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3463)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3434)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1170)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"ServletEventDispatcher: 2" Id=325 in WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@4b3794b3
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3463)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3434)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1170)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"ServletEventDispatcher: 3" Id=326 in WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@4b3794b3
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3463)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3434)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1170)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"Session-HouseKeeper-3ce6fdca-1" Id=329 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@651e4015
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1182)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"qtp726454768-331-acceptor-0@d68b154-0.0.0.0:8445@6369d271{SSL, (ssl, http/1.1)}{0.0.0.0:8445}" Id=331 in RUNNABLE (running in native)
    at java.base@17.0.7/sun.nio.ch.Net.accept(Native Method)
    at java.base@17.0.7/sun.nio.ch.ServerSocketChannelImpl.implAccept(ServerSocketChannelImpl.java:425)
    at java.base@17.0.7/sun.nio.ch.ServerSocketChannelImpl.accept(ServerSocketChannelImpl.java:391)
    at org.eclipse.jetty.server.ServerConnector.accept(ServerConnector.java:388)
    at org.eclipse.jetty.server.AbstractConnector$Acceptor.run(AbstractConnector.java:704)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

    Locked synchronizers: count = 1
      - java.util.concurrent.locks.ReentrantLock$NonfairSync@369f2905

"qtp726454768-334-acceptor-0@578cf2ed-default@15f03b01{HTTP/1.1, (http/1.1)}{0.0.0.0:8081}" Id=334 in RUNNABLE (running in native)
    at java.base@17.0.7/sun.nio.ch.Net.accept(Native Method)
    at java.base@17.0.7/sun.nio.ch.ServerSocketChannelImpl.implAccept(ServerSocketChannelImpl.java:425)
    at java.base@17.0.7/sun.nio.ch.ServerSocketChannelImpl.accept(ServerSocketChannelImpl.java:391)
    at org.eclipse.jetty.server.ServerConnector.accept(ServerConnector.java:388)
    at org.eclipse.jetty.server.AbstractConnector$Acceptor.run(AbstractConnector.java:704)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

    Locked synchronizers: count = 1
      - java.util.concurrent.locks.ReentrantLock$NonfairSync@41fab169

"SocketListener(172-17-0-1.local.)" Id=363 in RUNNABLE
    at java.base@17.0.7/sun.nio.ch.DatagramChannelImpl.receive0(Native Method)
    at java.base@17.0.7/sun.nio.ch.DatagramChannelImpl.receiveIntoNativeBuffer(DatagramChannelImpl.java:750)
    at java.base@17.0.7/sun.nio.ch.DatagramChannelImpl.receive(DatagramChannelImpl.java:728)
    at java.base@17.0.7/sun.nio.ch.DatagramChannelImpl.trustedBlockingReceive(DatagramChannelImpl.java:666)
    at java.base@17.0.7/sun.nio.ch.DatagramChannelImpl.blockingReceive(DatagramChannelImpl.java:635)
    at java.base@17.0.7/sun.nio.ch.DatagramSocketAdaptor.receive(DatagramSocketAdaptor.java:240)
    at java.base@17.0.7/java.net.DatagramSocket.receive(DatagramSocket.java:710)
    at javax.jmdns.impl.SocketListener.run(SocketListener.java:57)

    Locked synchronizers: count = 1
      - java.util.concurrent.locks.ReentrantLock$NonfairSync@56735fab

"SocketListener(server-goerentz-internal.local.)" Id=368 in RUNNABLE
    at java.base@17.0.7/sun.nio.ch.DatagramChannelImpl.receive0(Native Method)
    at java.base@17.0.7/sun.nio.ch.DatagramChannelImpl.receiveIntoNativeBuffer(DatagramChannelImpl.java:750)
    at java.base@17.0.7/sun.nio.ch.DatagramChannelImpl.receive(DatagramChannelImpl.java:728)
    at java.base@17.0.7/sun.nio.ch.DatagramChannelImpl.trustedBlockingReceive(DatagramChannelImpl.java:666)
    at java.base@17.0.7/sun.nio.ch.DatagramChannelImpl.blockingReceive(DatagramChannelImpl.java:635)
    at java.base@17.0.7/sun.nio.ch.DatagramSocketAdaptor.receive(DatagramSocketAdaptor.java:240)
    at java.base@17.0.7/java.net.DatagramSocket.receive(DatagramSocket.java:710)
    at javax.jmdns.impl.SocketListener.run(SocketListener.java:57)

    Locked synchronizers: count = 1
      - java.util.concurrent.locks.ReentrantLock$NonfairSync@6f497fe

"SocketListener(172-18-0-1.local.)" Id=374 in RUNNABLE
    at java.base@17.0.7/sun.nio.ch.DatagramChannelImpl.receive0(Native Method)
    at java.base@17.0.7/sun.nio.ch.DatagramChannelImpl.receiveIntoNativeBuffer(DatagramChannelImpl.java:750)
    at java.base@17.0.7/sun.nio.ch.DatagramChannelImpl.receive(DatagramChannelImpl.java:728)
    at java.base@17.0.7/sun.nio.ch.DatagramChannelImpl.trustedBlockingReceive(DatagramChannelImpl.java:666)
    at java.base@17.0.7/sun.nio.ch.DatagramChannelImpl.blockingReceive(DatagramChannelImpl.java:635)
    at java.base@17.0.7/sun.nio.ch.DatagramSocketAdaptor.receive(DatagramSocketAdaptor.java:240)
    at java.base@17.0.7/java.net.DatagramSocket.receive(DatagramSocket.java:710)
    at javax.jmdns.impl.SocketListener.run(SocketListener.java:57)

    Locked synchronizers: count = 1
      - java.util.concurrent.locks.ReentrantLock$NonfairSync@434a8c2f

"OH-karaf-addons-1" Id=387 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@638ef7b
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1182)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"Connector-Scheduler-15f03b01-1" Id=397 in WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@11042067
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3463)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3434)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1170)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"pool-15-thread-1" Id=415 in WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@6d1d07c8
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3463)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3434)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1170)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"OH-eventexecutor-1" Id=437 in WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@5a968dd8
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3463)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3434)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623)
    at java.base@17.0.7/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"OH-eventwatcher-1" Id=438 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@581c9f29
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1182)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"pool-6-thread-1" Id=439 in WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@8ca9204
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3463)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3434)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623)
    at java.base@17.0.7/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"OH-ruleengine-1" Id=448 in WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@54f7ef5
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3463)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3434)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1170)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"OH-lsp-1" Id=454 in RUNNABLE (running in native)
    at java.base@17.0.7/sun.nio.ch.Net.accept(Native Method)
    at java.base@17.0.7/sun.nio.ch.NioSocketImpl.accept(NioSocketImpl.java:755)
    at java.base@17.0.7/java.net.ServerSocket.implAccept(ServerSocket.java:686)
    at java.base@17.0.7/java.net.ServerSocket.platformImplAccept(ServerSocket.java:652)
    at java.base@17.0.7/java.net.ServerSocket.implAccept(ServerSocket.java:628)
    at java.base@17.0.7/java.net.ServerSocket.implAccept(ServerSocket.java:585)
    at java.base@17.0.7/java.net.ServerSocket.accept(ServerSocket.java:538)
    at org.openhab.core.model.lsp.internal.ModelServer.listen(ModelServer.java:105)
    at org.openhab.core.model.lsp.internal.ModelServer.lambda$0(ModelServer.java:84)
    at org.openhab.core.model.lsp.internal.ModelServer$$Lambda$980/0x00000007c0bc6a58.run(Unknown Source)
    at java.base@17.0.7/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
    at java.base@17.0.7/java.util.concurrent.FutureTask.run(FutureTask.java:264)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

    Locked synchronizers: count = 2
      - java.util.concurrent.ThreadPoolExecutor$Worker@1356d0d0
      - java.util.concurrent.locks.ReentrantLock$NonfairSync@6e65073d

"ServletModel-20-468" Id=468 in RUNNABLE (running in native)
    at java.base@17.0.7/sun.nio.ch.EPoll.wait(Native Method)
    at java.base@17.0.7/sun.nio.ch.EPollSelectorImpl.doSelect(EPollSelectorImpl.java:118)
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - locked sun.nio.ch.Util$2@552cc644
      - locked sun.nio.ch.EPollSelectorImpl@25463803
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.select(SelectorImpl.java:146)
    at org.eclipse.jetty.io.ManagedSelector.nioSelect(ManagedSelector.java:183)
    at org.eclipse.jetty.io.ManagedSelector.select(ManagedSelector.java:190)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.select(ManagedSelector.java:606)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.produce(ManagedSelector.java:543)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produceTask(EatWhatYouKill.java:362)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:186)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:173)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produce(EatWhatYouKill.java:137)
    at org.eclipse.jetty.io.ManagedSelector$$Lambda$719/0x00000007c06312c0.run(Unknown Source)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

    Locked monitors: count = 2
      - sun.nio.ch.Util$2@552cc644 locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - sun.nio.ch.EPollSelectorImpl@25463803 locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)

"ServletModel-20-469" Id=469 in RUNNABLE (running in native)
    at java.base@17.0.7/sun.nio.ch.EPoll.wait(Native Method)
    at java.base@17.0.7/sun.nio.ch.EPollSelectorImpl.doSelect(EPollSelectorImpl.java:118)
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - locked sun.nio.ch.Util$2@55865cb8
      - locked sun.nio.ch.EPollSelectorImpl@264b6232
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.select(SelectorImpl.java:146)
    at org.eclipse.jetty.io.ManagedSelector.nioSelect(ManagedSelector.java:183)
    at org.eclipse.jetty.io.ManagedSelector.select(ManagedSelector.java:190)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.select(ManagedSelector.java:606)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.produce(ManagedSelector.java:543)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produceTask(EatWhatYouKill.java:362)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:186)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:173)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produce(EatWhatYouKill.java:137)
    at org.eclipse.jetty.io.ManagedSelector$$Lambda$719/0x00000007c06312c0.run(Unknown Source)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

    Locked monitors: count = 2
      - sun.nio.ch.Util$2@55865cb8 locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - sun.nio.ch.EPollSelectorImpl@264b6232 locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)

"ServletModel-20-470" Id=470 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@15669e59
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"ServletModel-20-471" Id=471 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@15669e59
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"ServletModel-20-472" Id=472 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@15669e59
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"ServletModel-20-473" Id=473 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@15669e59
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"ServletModel-20-474" Id=474 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@15669e59
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"ServletModel-20-475" Id=475 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@15669e59
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"HttpClient@2f4ccc15-504" Id=504 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@1f43ad6
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"HttpClient@2f4ccc15-505" Id=505 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@1f43ad6
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"HttpClient@2f4ccc15-506" Id=506 in RUNNABLE
    at java.base@17.0.7/sun.nio.ch.EPoll.wait(Native Method)
    at java.base@17.0.7/sun.nio.ch.EPollSelectorImpl.doSelect(EPollSelectorImpl.java:118)
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - locked sun.nio.ch.Util$2@66f0c5cc
      - locked sun.nio.ch.EPollSelectorImpl@1bd761d2
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.select(SelectorImpl.java:146)
    at org.eclipse.jetty.io.ManagedSelector.nioSelect(ManagedSelector.java:183)
    at org.eclipse.jetty.io.ManagedSelector.select(ManagedSelector.java:190)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.select(ManagedSelector.java:606)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.produce(ManagedSelector.java:543)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produceTask(EatWhatYouKill.java:362)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:186)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:173)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:131)
    at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:409)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

    Locked monitors: count = 2
      - sun.nio.ch.Util$2@66f0c5cc locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - sun.nio.ch.EPollSelectorImpl@1bd761d2 locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)

"HttpClient@2f4ccc15-507" Id=507 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@1f43ad6
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"HttpClient@2f4ccc15-508" Id=508 in TIMED_WAITING on lock=java.util.concurrent.SynchronousQueue$TransferStack@3a62a0a7
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.SynchronousQueue$TransferStack.transfer(SynchronousQueue.java:401)
    at java.base@17.0.7/java.util.concurrent.SynchronousQueue.poll(SynchronousQueue.java:903)
    at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.reservedWait(ReservedThreadExecutor.java:324)
    at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:399)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"HttpClient@2f4ccc15-509" Id=509 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@1f43ad6
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"HttpClient@2f4ccc15-510" Id=510 in RUNNABLE (running in native)
    at java.base@17.0.7/sun.nio.ch.EPoll.wait(Native Method)
    at java.base@17.0.7/sun.nio.ch.EPollSelectorImpl.doSelect(EPollSelectorImpl.java:118)
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - locked sun.nio.ch.Util$2@1ae00fb7
      - locked sun.nio.ch.EPollSelectorImpl@7ade6782
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.select(SelectorImpl.java:146)
    at org.eclipse.jetty.io.ManagedSelector.nioSelect(ManagedSelector.java:183)
    at org.eclipse.jetty.io.ManagedSelector.select(ManagedSelector.java:190)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.select(ManagedSelector.java:606)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.produce(ManagedSelector.java:543)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produceTask(EatWhatYouKill.java:362)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:186)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:173)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:131)
    at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:409)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

    Locked monitors: count = 2
      - sun.nio.ch.Util$2@1ae00fb7 locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - sun.nio.ch.EPollSelectorImpl@7ade6782 locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)

"HttpClient@2f4ccc15-511" Id=511 in TIMED_WAITING on lock=java.util.concurrent.SynchronousQueue$TransferStack@3a62a0a7
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.SynchronousQueue$TransferStack.transfer(SynchronousQueue.java:401)
    at java.base@17.0.7/java.util.concurrent.SynchronousQueue.poll(SynchronousQueue.java:903)
    at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.reservedWait(ReservedThreadExecutor.java:324)
    at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:399)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"OH-httpClient-common-514" Id=514 in TIMED_WAITING on lock=java.util.concurrent.SynchronousQueue$TransferStack@5aa740fa
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.SynchronousQueue$TransferStack.transfer(SynchronousQueue.java:401)
    at java.base@17.0.7/java.util.concurrent.SynchronousQueue.poll(SynchronousQueue.java:903)
    at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.reservedWait(ReservedThreadExecutor.java:324)
    at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:399)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"OH-httpClient-common-515" Id=515 in RUNNABLE
    at java.base@17.0.7/sun.nio.ch.EPoll.wait(Native Method)
    at java.base@17.0.7/sun.nio.ch.EPollSelectorImpl.doSelect(EPollSelectorImpl.java:118)
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - locked sun.nio.ch.Util$2@75efa491
      - locked sun.nio.ch.EPollSelectorImpl@6fe8902c
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.select(SelectorImpl.java:146)
    at org.eclipse.jetty.io.ManagedSelector.nioSelect(ManagedSelector.java:183)
    at org.eclipse.jetty.io.ManagedSelector.select(ManagedSelector.java:190)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.select(ManagedSelector.java:606)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.produce(ManagedSelector.java:543)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produceTask(EatWhatYouKill.java:362)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:186)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:173)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:131)
    at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:409)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

    Locked monitors: count = 2
      - sun.nio.ch.Util$2@75efa491 locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - sun.nio.ch.EPollSelectorImpl@6fe8902c locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)

"OH-httpClient-common-516" Id=516 in RUNNABLE (running in native)
    at java.base@17.0.7/sun.nio.ch.EPoll.wait(Native Method)
    at java.base@17.0.7/sun.nio.ch.EPollSelectorImpl.doSelect(EPollSelectorImpl.java:118)
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - locked sun.nio.ch.Util$2@66830824
      - locked sun.nio.ch.EPollSelectorImpl@6c3249e
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.select(SelectorImpl.java:146)
    at org.eclipse.jetty.io.ManagedSelector.nioSelect(ManagedSelector.java:183)
    at org.eclipse.jetty.io.ManagedSelector.select(ManagedSelector.java:190)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.select(ManagedSelector.java:606)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.produce(ManagedSelector.java:543)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produceTask(EatWhatYouKill.java:362)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:186)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:173)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produce(EatWhatYouKill.java:137)
    at org.eclipse.jetty.io.ManagedSelector$$Lambda$719/0x00000007c06312c0.run(Unknown Source)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

    Locked monitors: count = 2
      - sun.nio.ch.Util$2@66830824 locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - sun.nio.ch.EPollSelectorImpl@6c3249e locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)

"OH-httpClient-common-517" Id=517 in RUNNABLE (running in native)
    at java.base@17.0.7/sun.nio.ch.EPoll.wait(Native Method)
    at java.base@17.0.7/sun.nio.ch.EPollSelectorImpl.doSelect(EPollSelectorImpl.java:118)
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - locked sun.nio.ch.Util$2@e80c3fb
      - locked sun.nio.ch.EPollSelectorImpl@25c24bf7
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.select(SelectorImpl.java:146)
    at org.eclipse.jetty.io.ManagedSelector.nioSelect(ManagedSelector.java:183)
    at org.eclipse.jetty.io.ManagedSelector.select(ManagedSelector.java:190)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.select(ManagedSelector.java:606)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.produce(ManagedSelector.java:543)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produceTask(EatWhatYouKill.java:362)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:186)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:173)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produce(EatWhatYouKill.java:137)
    at org.eclipse.jetty.io.ManagedSelector$$Lambda$719/0x00000007c06312c0.run(Unknown Source)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

    Locked monitors: count = 2
      - sun.nio.ch.Util$2@e80c3fb locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - sun.nio.ch.EPollSelectorImpl@25c24bf7 locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)

"OH-httpClient-common-518" Id=518 in RUNNABLE (running in native)
    at java.base@17.0.7/sun.nio.ch.EPoll.wait(Native Method)
    at java.base@17.0.7/sun.nio.ch.EPollSelectorImpl.doSelect(EPollSelectorImpl.java:118)
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - locked sun.nio.ch.Util$2@5bfeb543
      - locked sun.nio.ch.EPollSelectorImpl@44068d76
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.select(SelectorImpl.java:146)
    at org.eclipse.jetty.io.ManagedSelector.nioSelect(ManagedSelector.java:183)
    at org.eclipse.jetty.io.ManagedSelector.select(ManagedSelector.java:190)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.select(ManagedSelector.java:606)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.produce(ManagedSelector.java:543)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produceTask(EatWhatYouKill.java:362)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:186)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:173)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:131)
    at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:409)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

    Locked monitors: count = 2
      - sun.nio.ch.Util$2@5bfeb543 locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - sun.nio.ch.EPollSelectorImpl@44068d76 locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)

"OH-httpClient-common-519" Id=519 in TIMED_WAITING on lock=java.util.concurrent.SynchronousQueue$TransferStack@5aa740fa
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.SynchronousQueue$TransferStack.transfer(SynchronousQueue.java:401)
    at java.base@17.0.7/java.util.concurrent.SynchronousQueue.poll(SynchronousQueue.java:903)
    at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.reservedWait(ReservedThreadExecutor.java:324)
    at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:399)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"OH-httpClient-common-520" Id=520 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@4c110116
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"OH-httpClient-common-521" Id=521 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@4c110116
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"OH-httpClient-common-522" Id=522 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@4c110116
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"OH-httpClient-common-523" Id=523 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@4c110116
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"OH-thingHandler-1" Id=526 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@7b3b7690
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.poll(ScheduledThreadPoolExecutor.java:1218)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.poll(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1061)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"HttpClient@77e1bf50-scheduler-1" Id=529 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@55c18019
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1182)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"OH-thingHandler-2" Id=530 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@7b3b7690
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.poll(ScheduledThreadPoolExecutor.java:1218)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.poll(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1061)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"HttpClient@2f4ccc15-scheduler-1" Id=532 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@25dccaea
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1182)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"OH-thingHandler-3" Id=533 in WAITING on lock=java.util.concurrent.CountDownLatch$Sync@41ae13a0
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.park(LockSupport.java:211)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer.acquire(AbstractQueuedSynchronizer.java:715)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer.acquireSharedInterruptibly(AbstractQueuedSynchronizer.java:1047)
    at java.base@17.0.7/java.util.concurrent.CountDownLatch.await(CountDownLatch.java:230)
    at org.eclipse.jetty.client.util.FutureResponseListener.get(FutureResponseListener.java:100)
    at org.eclipse.jetty.client.HttpRequest.send(HttpRequest.java:730)
    at org.openhab.binding.easyfamily.internal.httpclient.EasyFamilyHttpClient.sendMsg(EasyFamilyHttpClient.java:106)
    at org.openhab.binding.easyfamily.internal.EasyFamilyHandler.getData(EasyFamilyHandler.java:650)
      - locked org.openhab.binding.easyfamily.internal.httpclient.EasyFamilyHttpClient@618176f6
    at org.openhab.binding.easyfamily.internal.EasyFamilyHandler$PollForData.run(EasyFamilyHandler.java:1042)
    at java.base@17.0.7/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
    at java.base@17.0.7/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:305)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

    Locked synchronizers: count = 1
      - java.util.concurrent.ThreadPoolExecutor$Worker@59b8a902

    Locked monitors: count = 1
      - org.openhab.binding.easyfamily.internal.httpclient.EasyFamilyHttpClient@618176f6 locked at
          8 org.openhab.binding.easyfamily.internal.EasyFamilyHandler.getData(EasyFamilyHandler.java:650)

"OH-items-1" Id=536 in BLOCKED on lock=java.util.HashMap@18b692c4
     owned by OH-items-5 Id=564
    at org.openhab.core.persistence.internal.PersistenceManagerImpl.handleStateEvent(PersistenceManagerImpl.java:142)
    at org.openhab.core.persistence.internal.PersistenceManagerImpl.stateUpdated(PersistenceManagerImpl.java:478)
    at org.openhab.core.items.GenericItem.lambda$1(GenericItem.java:257)
    at org.openhab.core.items.GenericItem$$Lambda$1107/0x00000007c0c7c498.run(Unknown Source)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

    Locked synchronizers: count = 1
      - java.util.concurrent.ThreadPoolExecutor$Worker@5e10a110

"OH-items-2" Id=537 in TIMED_WAITING on lock=java.util.concurrent.SynchronousQueue$TransferStack@5b7e90a2
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.SynchronousQueue$TransferStack.transfer(SynchronousQueue.java:401)
    at java.base@17.0.7/java.util.concurrent.SynchronousQueue.poll(SynchronousQueue.java:903)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1061)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"OH-items-3" Id=538 in TIMED_WAITING on lock=java.util.concurrent.SynchronousQueue$TransferStack@5b7e90a2
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.SynchronousQueue$TransferStack.transfer(SynchronousQueue.java:401)
    at java.base@17.0.7/java.util.concurrent.SynchronousQueue.poll(SynchronousQueue.java:903)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1061)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"OH-thingHandler-4" Id=541 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@7b3b7690
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.poll(ScheduledThreadPoolExecutor.java:1223)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.poll(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1061)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"HttpClient@63acd8a6-542" Id=542 in RUNNABLE (running in native)
    at java.base@17.0.7/sun.nio.ch.EPoll.wait(Native Method)
    at java.base@17.0.7/sun.nio.ch.EPollSelectorImpl.doSelect(EPollSelectorImpl.java:118)
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - locked sun.nio.ch.Util$2@31a27bfd
      - locked sun.nio.ch.EPollSelectorImpl@a05d7a
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.select(SelectorImpl.java:146)
    at org.eclipse.jetty.io.ManagedSelector.nioSelect(ManagedSelector.java:183)
    at org.eclipse.jetty.io.ManagedSelector.select(ManagedSelector.java:190)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.select(ManagedSelector.java:606)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.produce(ManagedSelector.java:543)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produceTask(EatWhatYouKill.java:362)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:186)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:173)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produce(EatWhatYouKill.java:137)
    at org.eclipse.jetty.io.ManagedSelector$$Lambda$719/0x00000007c06312c0.run(Unknown Source)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

    Locked monitors: count = 2
      - sun.nio.ch.Util$2@31a27bfd locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - sun.nio.ch.EPollSelectorImpl@a05d7a locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)

"HttpClient@63acd8a6-543" Id=543 in RUNNABLE (running in native)
    at java.base@17.0.7/sun.nio.ch.EPoll.wait(Native Method)
    at java.base@17.0.7/sun.nio.ch.EPollSelectorImpl.doSelect(EPollSelectorImpl.java:118)
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - locked sun.nio.ch.Util$2@2caccf18
      - locked sun.nio.ch.EPollSelectorImpl@378b1af
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.select(SelectorImpl.java:146)
    at org.eclipse.jetty.io.ManagedSelector.nioSelect(ManagedSelector.java:183)
    at org.eclipse.jetty.io.ManagedSelector.select(ManagedSelector.java:190)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.select(ManagedSelector.java:606)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.produce(ManagedSelector.java:543)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produceTask(EatWhatYouKill.java:362)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:186)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:173)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produce(EatWhatYouKill.java:137)
    at org.eclipse.jetty.io.ManagedSelector$$Lambda$719/0x00000007c06312c0.run(Unknown Source)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

    Locked monitors: count = 2
      - sun.nio.ch.Util$2@2caccf18 locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - sun.nio.ch.EPollSelectorImpl@378b1af locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)

"HttpClient@63acd8a6-544" Id=544 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@21275fae
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"HttpClient@63acd8a6-545" Id=545 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@21275fae
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"HttpClient@63acd8a6-546" Id=546 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@21275fae
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"HttpClient@63acd8a6-547" Id=547 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@21275fae
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"HttpClient@63acd8a6-548" Id=548 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@21275fae
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"HttpClient@63acd8a6-549" Id=549 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@21275fae
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"OH-scriptwatcher-1" Id=552 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@236e9d7d
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1182)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"OH-rulesRefresher-1" Id=555 in WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@69f04207
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3463)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3434)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1170)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"OH-items-4" Id=556 in BLOCKED on lock=java.util.HashMap@18b692c4
     owned by OH-items-5 Id=564
    at org.openhab.core.persistence.internal.PersistenceManagerImpl.handleStateEvent(PersistenceManagerImpl.java:142)
    at org.openhab.core.persistence.internal.PersistenceManagerImpl.stateUpdated(PersistenceManagerImpl.java:478)
    at org.openhab.core.items.GenericItem.lambda$1(GenericItem.java:257)
    at org.openhab.core.items.GenericItem$$Lambda$1107/0x00000007c0c7c498.run(Unknown Source)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

    Locked synchronizers: count = 1
      - java.util.concurrent.ThreadPoolExecutor$Worker@3143e2f8

"OH-thingHandler-5" Id=559 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@7b3b7690
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.poll(ScheduledThreadPoolExecutor.java:1218)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.poll(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1061)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"RRD4J Sync-ThreadPool for org.rrd4j.core.RrdSyncThreadPool@21bdb6f0 [Thread-1]" Id=561 in WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@2fc3735f
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3463)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3434)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1177)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"RRD4J Sync-ThreadPool for org.rrd4j.core.RrdSyncThreadPool@21bdb6f0 [Thread-2]" Id=562 in WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@2fc3735f
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3463)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3434)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1177)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"RRD4J Sync-ThreadPool for org.rrd4j.core.RrdSyncThreadPool@21bdb6f0 [Thread-3]" Id=563 in WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@2fc3735f
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3463)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3434)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1177)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"OH-items-5" Id=564 in RUNNABLE
    at org.openhab.core.persistence.internal.PersistenceManagerImpl.handleStateEvent(PersistenceManagerImpl.java:142)
      - locked java.util.HashMap@18b692c4
    at org.openhab.core.persistence.internal.PersistenceManagerImpl.stateUpdated(PersistenceManagerImpl.java:478)
    at org.openhab.core.items.GenericItem.lambda$1(GenericItem.java:257)
    at org.openhab.core.items.GenericItem$$Lambda$1107/0x00000007c0c7c498.run(Unknown Source)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

    Locked synchronizers: count = 1
      - java.util.concurrent.ThreadPoolExecutor$Worker@3204d1ed

    Locked monitors: count = 1
      - java.util.HashMap@18b692c4 locked at
          0 org.openhab.core.persistence.internal.PersistenceManagerImpl.handleStateEvent(PersistenceManagerImpl.java:142)

"RRD4J Sync-ThreadPool for org.rrd4j.core.RrdSyncThreadPool@21bdb6f0 [Thread-4]" Id=566 in WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@2fc3735f
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3463)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3434)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1177)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"RRD4J Sync-ThreadPool for org.rrd4j.core.RrdSyncThreadPool@21bdb6f0 [Thread-5]" Id=567 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@2fc3735f
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1182)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"RRD4J Sync-ThreadPool for org.rrd4j.core.RrdSyncThreadPool@21bdb6f0 [Thread-6]" Id=568 in WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@2fc3735f
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3463)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3434)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1177)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"OH-RRD4j-1" Id=586 in RUNNABLE
    at java.base@17.0.7/java.lang.StringLatin1.hashCode(StringLatin1.java:196)
    at java.base@17.0.7/java.lang.String.hashCode(String.java:2340)
    at java.base@17.0.7/java.util.ImmutableCollections$SetN.probe(ImmutableCollections.java:1007)
    at java.base@17.0.7/java.util.ImmutableCollections$SetN.contains(ImmutableCollections.java:938)
    at org.openhab.persistence.rrd4j.internal.RRD4jPersistenceService.isSupportedItemType(RRD4jPersistenceService.java:470)
    at org.openhab.persistence.rrd4j.internal.RRD4jPersistenceService.store(RRD4jPersistenceService.java:149)
      - locked org.openhab.persistence.rrd4j.internal.RRD4jPersistenceService@5afffbb1
    at org.openhab.persistence.rrd4j.internal.RRD4jPersistenceService.lambda$0(RRD4jPersistenceService.java:233)
    at org.openhab.persistence.rrd4j.internal.RRD4jPersistenceService$$Lambda$1221/0x00000007c0d10af0.run(Unknown Source)
    at java.base@17.0.7/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
    at java.base@17.0.7/java.util.concurrent.FutureTask.run(FutureTask.java:264)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

    Locked synchronizers: count = 1
      - java.util.concurrent.ThreadPoolExecutor$Worker@3139e441

    Locked monitors: count = 1
      - org.openhab.persistence.rrd4j.internal.RRD4jPersistenceService@5afffbb1 locked at
          5 org.openhab.persistence.rrd4j.internal.RRD4jPersistenceService.store(RRD4jPersistenceService.java:149)

"OH-RRD4j-2" Id=587 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@34029326
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1182)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"OH-RRD4j-3" Id=589 in WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@34029326
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3463)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3434)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1177)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-590" Id=590 in RUNNABLE (running in native)
    at java.base@17.0.7/sun.nio.ch.EPoll.wait(Native Method)
    at java.base@17.0.7/sun.nio.ch.EPollSelectorImpl.doSelect(EPollSelectorImpl.java:118)
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - locked sun.nio.ch.Util$2@63d075ae
      - locked sun.nio.ch.EPollSelectorImpl@8dbf3a9
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.select(SelectorImpl.java:146)
    at org.eclipse.jetty.io.ManagedSelector.nioSelect(ManagedSelector.java:183)
    at org.eclipse.jetty.io.ManagedSelector.select(ManagedSelector.java:190)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.select(ManagedSelector.java:606)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.produce(ManagedSelector.java:543)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produceTask(EatWhatYouKill.java:362)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:186)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:173)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produce(EatWhatYouKill.java:137)
    at org.eclipse.jetty.io.ManagedSelector$$Lambda$719/0x00000007c06312c0.run(Unknown Source)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

    Locked monitors: count = 2
      - sun.nio.ch.Util$2@63d075ae locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - sun.nio.ch.EPollSelectorImpl@8dbf3a9 locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)

"EasyFamilyHttpClient-591" Id=591 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@3c362066
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-592" Id=592 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@3c362066
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-593" Id=593 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@3c362066
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-594" Id=594 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@3c362066
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-595" Id=595 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@3c362066
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-596" Id=596 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@3c362066
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-597" Id=597 in RUNNABLE (running in native)
    at java.base@17.0.7/sun.nio.ch.EPoll.wait(Native Method)
    at java.base@17.0.7/sun.nio.ch.EPollSelectorImpl.doSelect(EPollSelectorImpl.java:118)
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - locked sun.nio.ch.Util$2@1c0d0f65
      - locked sun.nio.ch.EPollSelectorImpl@7c10744f
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.select(SelectorImpl.java:146)
    at org.eclipse.jetty.io.ManagedSelector.nioSelect(ManagedSelector.java:183)
    at org.eclipse.jetty.io.ManagedSelector.select(ManagedSelector.java:190)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.select(ManagedSelector.java:606)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.produce(ManagedSelector.java:543)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produceTask(EatWhatYouKill.java:362)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:186)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:173)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produce(EatWhatYouKill.java:137)
    at org.eclipse.jetty.io.ManagedSelector$$Lambda$719/0x00000007c06312c0.run(Unknown Source)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

    Locked monitors: count = 2
      - sun.nio.ch.Util$2@1c0d0f65 locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - sun.nio.ch.EPollSelectorImpl@7c10744f locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)

"pool-18-thread-1" Id=598 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@2b8186df
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1182)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-599" Id=599 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@26b9b6d9
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-600" Id=600 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@26b9b6d9
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-601" Id=601 in RUNNABLE (running in native)
    at java.base@17.0.7/sun.nio.ch.EPoll.wait(Native Method)
    at java.base@17.0.7/sun.nio.ch.EPollSelectorImpl.doSelect(EPollSelectorImpl.java:118)
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - locked sun.nio.ch.Util$2@52bbf34c
      - locked sun.nio.ch.EPollSelectorImpl@2df65758
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.select(SelectorImpl.java:146)
    at org.eclipse.jetty.io.ManagedSelector.nioSelect(ManagedSelector.java:183)
    at org.eclipse.jetty.io.ManagedSelector.select(ManagedSelector.java:190)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.select(ManagedSelector.java:606)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.produce(ManagedSelector.java:543)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produceTask(EatWhatYouKill.java:362)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:186)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:173)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produce(EatWhatYouKill.java:137)
    at org.eclipse.jetty.io.ManagedSelector$$Lambda$719/0x00000007c06312c0.run(Unknown Source)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

    Locked monitors: count = 2
      - sun.nio.ch.Util$2@52bbf34c locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - sun.nio.ch.EPollSelectorImpl@2df65758 locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)

"EasyFamilyHttpClient-602" Id=602 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@26b9b6d9
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-603" Id=603 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@26b9b6d9
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-604" Id=604 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@26b9b6d9
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-605" Id=605 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@26b9b6d9
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-606" Id=606 in RUNNABLE (running in native)
    at java.base@17.0.7/sun.nio.ch.EPoll.wait(Native Method)
    at java.base@17.0.7/sun.nio.ch.EPollSelectorImpl.doSelect(EPollSelectorImpl.java:118)
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - locked sun.nio.ch.Util$2@ae69b47
      - locked sun.nio.ch.EPollSelectorImpl@6182fbbc
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.select(SelectorImpl.java:146)
    at org.eclipse.jetty.io.ManagedSelector.nioSelect(ManagedSelector.java:183)
    at org.eclipse.jetty.io.ManagedSelector.select(ManagedSelector.java:190)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.select(ManagedSelector.java:606)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.produce(ManagedSelector.java:543)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produceTask(EatWhatYouKill.java:362)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:186)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:173)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produce(EatWhatYouKill.java:137)
    at org.eclipse.jetty.io.ManagedSelector$$Lambda$719/0x00000007c06312c0.run(Unknown Source)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

    Locked monitors: count = 2
      - sun.nio.ch.Util$2@ae69b47 locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - sun.nio.ch.EPollSelectorImpl@6182fbbc locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)

"pool-19-thread-1" Id=607 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@6cae5553
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1182)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-608" Id=608 in RUNNABLE (running in native)
    at java.base@17.0.7/sun.nio.ch.EPoll.wait(Native Method)
    at java.base@17.0.7/sun.nio.ch.EPollSelectorImpl.doSelect(EPollSelectorImpl.java:118)
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - locked sun.nio.ch.Util$2@737d9309
      - locked sun.nio.ch.EPollSelectorImpl@4dff0282
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.select(SelectorImpl.java:146)
    at org.eclipse.jetty.io.ManagedSelector.nioSelect(ManagedSelector.java:183)
    at org.eclipse.jetty.io.ManagedSelector.select(ManagedSelector.java:190)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.select(ManagedSelector.java:606)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.produce(ManagedSelector.java:543)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produceTask(EatWhatYouKill.java:362)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:186)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:173)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produce(EatWhatYouKill.java:137)
    at org.eclipse.jetty.io.ManagedSelector$$Lambda$719/0x00000007c06312c0.run(Unknown Source)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

    Locked monitors: count = 2
      - sun.nio.ch.Util$2@737d9309 locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - sun.nio.ch.EPollSelectorImpl@4dff0282 locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)

"EasyFamilyHttpClient-609" Id=609 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@3b565f9a
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-610" Id=610 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@3b565f9a
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-611" Id=611 in RUNNABLE (running in native)
    at java.base@17.0.7/sun.nio.ch.EPoll.wait(Native Method)
    at java.base@17.0.7/sun.nio.ch.EPollSelectorImpl.doSelect(EPollSelectorImpl.java:118)
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - locked sun.nio.ch.Util$2@25cc916e
      - locked sun.nio.ch.EPollSelectorImpl@3ab6bb05
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.select(SelectorImpl.java:146)
    at org.eclipse.jetty.io.ManagedSelector.nioSelect(ManagedSelector.java:183)
    at org.eclipse.jetty.io.ManagedSelector.select(ManagedSelector.java:190)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.select(ManagedSelector.java:606)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.produce(ManagedSelector.java:543)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produceTask(EatWhatYouKill.java:362)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:186)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:173)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produce(EatWhatYouKill.java:137)
    at org.eclipse.jetty.io.ManagedSelector$$Lambda$719/0x00000007c06312c0.run(Unknown Source)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

    Locked monitors: count = 2
      - sun.nio.ch.Util$2@25cc916e locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - sun.nio.ch.EPollSelectorImpl@3ab6bb05 locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)

"EasyFamilyHttpClient-612" Id=612 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@3b565f9a
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-613" Id=613 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@3b565f9a
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-614" Id=614 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@3b565f9a
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-615" Id=615 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@3b565f9a
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"pool-20-thread-1" Id=616 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@f6845c3
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1182)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-617" Id=617 in RUNNABLE (running in native)
    at java.base@17.0.7/sun.nio.ch.EPoll.wait(Native Method)
    at java.base@17.0.7/sun.nio.ch.EPollSelectorImpl.doSelect(EPollSelectorImpl.java:118)
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - locked sun.nio.ch.Util$2@64b613b
      - locked sun.nio.ch.EPollSelectorImpl@47cdd77d
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.select(SelectorImpl.java:146)
    at org.eclipse.jetty.io.ManagedSelector.nioSelect(ManagedSelector.java:183)
    at org.eclipse.jetty.io.ManagedSelector.select(ManagedSelector.java:190)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.select(ManagedSelector.java:606)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.produce(ManagedSelector.java:543)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produceTask(EatWhatYouKill.java:362)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:186)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:173)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produce(EatWhatYouKill.java:137)
    at org.eclipse.jetty.io.ManagedSelector$$Lambda$719/0x00000007c06312c0.run(Unknown Source)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

    Locked monitors: count = 2
      - sun.nio.ch.Util$2@64b613b locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - sun.nio.ch.EPollSelectorImpl@47cdd77d locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)

"EasyFamilyHttpClient-618" Id=618 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@482c2212
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-619" Id=619 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@482c2212
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-620" Id=620 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@482c2212
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-621" Id=621 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@482c2212
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-622" Id=622 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@482c2212
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-623" Id=623 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@482c2212
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-624" Id=624 in RUNNABLE (running in native)
    at java.base@17.0.7/sun.nio.ch.EPoll.wait(Native Method)
    at java.base@17.0.7/sun.nio.ch.EPollSelectorImpl.doSelect(EPollSelectorImpl.java:118)
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - locked sun.nio.ch.Util$2@5272089
      - locked sun.nio.ch.EPollSelectorImpl@58b8cbaf
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.select(SelectorImpl.java:146)
    at org.eclipse.jetty.io.ManagedSelector.nioSelect(ManagedSelector.java:183)
    at org.eclipse.jetty.io.ManagedSelector.select(ManagedSelector.java:190)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.select(ManagedSelector.java:606)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.produce(ManagedSelector.java:543)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produceTask(EatWhatYouKill.java:362)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:186)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:173)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produce(EatWhatYouKill.java:137)
    at org.eclipse.jetty.io.ManagedSelector$$Lambda$719/0x00000007c06312c0.run(Unknown Source)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

    Locked monitors: count = 2
      - sun.nio.ch.Util$2@5272089 locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - sun.nio.ch.EPollSelectorImpl@58b8cbaf locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)

"pool-21-thread-1" Id=625 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@71e789fa
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1182)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-642" Id=642 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@64ed5989
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-641" Id=641 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@712dd430
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-643" Id=643 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@64ed5989
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-644" Id=644 in RUNNABLE
    at java.base@17.0.7/sun.nio.ch.EPoll.wait(Native Method)
    at java.base@17.0.7/sun.nio.ch.EPollSelectorImpl.doSelect(EPollSelectorImpl.java:118)
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - locked sun.nio.ch.Util$2@7a960308
      - locked sun.nio.ch.EPollSelectorImpl@3773edf9
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.select(SelectorImpl.java:146)
    at org.eclipse.jetty.io.ManagedSelector.nioSelect(ManagedSelector.java:183)
    at org.eclipse.jetty.io.ManagedSelector.select(ManagedSelector.java:190)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.select(ManagedSelector.java:606)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.produce(ManagedSelector.java:543)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produceTask(EatWhatYouKill.java:362)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:186)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:173)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:131)
    at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:409)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

    Locked monitors: count = 2
      - sun.nio.ch.Util$2@7a960308 locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - sun.nio.ch.EPollSelectorImpl@3773edf9 locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)

"EasyFamilyHttpClient-645" Id=645 in TIMED_WAITING on lock=java.util.concurrent.SynchronousQueue$TransferStack@7f02fb48
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.SynchronousQueue$TransferStack.transfer(SynchronousQueue.java:401)
    at java.base@17.0.7/java.util.concurrent.SynchronousQueue.poll(SynchronousQueue.java:903)
    at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.reservedWait(ReservedThreadExecutor.java:324)
    at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:399)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-646" Id=646 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@7bed5877
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-647" Id=647 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@64ed5989
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-648" Id=648 in RUNNABLE (running in native)
    at java.base@17.0.7/sun.nio.ch.EPoll.wait(Native Method)
    at java.base@17.0.7/sun.nio.ch.EPollSelectorImpl.doSelect(EPollSelectorImpl.java:118)
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - locked sun.nio.ch.Util$2@3be9075e
      - locked sun.nio.ch.EPollSelectorImpl@13dd29ff
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.select(SelectorImpl.java:146)
    at org.eclipse.jetty.io.ManagedSelector.nioSelect(ManagedSelector.java:183)
    at org.eclipse.jetty.io.ManagedSelector.select(ManagedSelector.java:190)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.select(ManagedSelector.java:606)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.produce(ManagedSelector.java:543)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produceTask(EatWhatYouKill.java:362)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:186)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:173)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:131)
    at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:409)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

    Locked monitors: count = 2
      - sun.nio.ch.Util$2@3be9075e locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - sun.nio.ch.EPollSelectorImpl@13dd29ff locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)

"EasyFamilyHttpClient-649" Id=649 in TIMED_WAITING on lock=java.util.concurrent.SynchronousQueue$TransferStack@64fe8f6e
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.SynchronousQueue$TransferStack.transfer(SynchronousQueue.java:401)
    at java.base@17.0.7/java.util.concurrent.SynchronousQueue.poll(SynchronousQueue.java:903)
    at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.reservedWait(ReservedThreadExecutor.java:324)
    at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:399)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-650" Id=650 in RUNNABLE (running in native)
    at java.base@17.0.7/sun.nio.ch.EPoll.wait(Native Method)
    at java.base@17.0.7/sun.nio.ch.EPollSelectorImpl.doSelect(EPollSelectorImpl.java:118)
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - locked sun.nio.ch.Util$2@17deb758
      - locked sun.nio.ch.EPollSelectorImpl@6d36f68d
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.select(SelectorImpl.java:146)
    at org.eclipse.jetty.io.ManagedSelector.nioSelect(ManagedSelector.java:183)
    at org.eclipse.jetty.io.ManagedSelector.select(ManagedSelector.java:190)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.select(ManagedSelector.java:606)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.produce(ManagedSelector.java:543)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produceTask(EatWhatYouKill.java:362)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:186)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:173)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:131)
    at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:409)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

    Locked monitors: count = 2
      - sun.nio.ch.Util$2@17deb758 locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - sun.nio.ch.EPollSelectorImpl@6d36f68d locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)

"EasyFamilyHttpClient-651" Id=651 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@7bed5877
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-652" Id=652 in RUNNABLE
    at java.base@17.0.7/sun.nio.ch.EventFD.set0(Native Method)
    at java.base@17.0.7/sun.nio.ch.EventFD.set(EventFD.java:48)
    at java.base@17.0.7/sun.nio.ch.EPollSelectorImpl.wakeup(EPollSelectorImpl.java:251)
      - locked java.lang.Object@e1a5e10
    at org.eclipse.jetty.io.ManagedSelector.submit(ManagedSelector.java:305)
    at org.eclipse.jetty.io.ManagedSelector.submit(ManagedSelector.java:280)
    at org.eclipse.jetty.io.ChannelEndPoint.changeInterests(ChannelEndPoint.java:425)
    at org.eclipse.jetty.io.ChannelEndPoint.needsFillInterest(ChannelEndPoint.java:308)
    at org.eclipse.jetty.io.AbstractEndPoint$1.needsFillInterest(AbstractEndPoint.java:45)
    at org.eclipse.jetty.io.FillInterest.tryRegister(FillInterest.java:83)
    at org.eclipse.jetty.io.AbstractEndPoint.tryFillInterested(AbstractEndPoint.java:369)
    at org.eclipse.jetty.io.AbstractConnection.tryFillInterested(AbstractConnection.java:144)
    at org.eclipse.jetty.io.ssl.SslConnection$DecryptedEndPoint.ensureFillInterested(SslConnection.java:1385)
    at org.eclipse.jetty.io.ssl.SslConnection$DecryptedEndPoint.needsFillInterest(SslConnection.java:924)
    at org.eclipse.jetty.io.AbstractEndPoint$1.needsFillInterest(AbstractEndPoint.java:45)
    at org.eclipse.jetty.io.FillInterest.tryRegister(FillInterest.java:83)
    at org.eclipse.jetty.io.FillInterest.register(FillInterest.java:55)
    at org.eclipse.jetty.io.AbstractEndPoint.fillInterested(AbstractEndPoint.java:362)
    at org.eclipse.jetty.io.AbstractConnection.fillInterested(AbstractConnection.java:134)
    at org.eclipse.jetty.client.http.HttpReceiverOverHTTP.fillInterested(HttpReceiverOverHTTP.java:253)
    at org.eclipse.jetty.client.http.HttpReceiverOverHTTP.process(HttpReceiverOverHTTP.java:179)
    at org.eclipse.jetty.client.http.HttpReceiverOverHTTP.receive(HttpReceiverOverHTTP.java:80)
    at org.eclipse.jetty.client.http.HttpChannelOverHTTP.receive(HttpChannelOverHTTP.java:131)
    at org.eclipse.jetty.client.http.HttpConnectionOverHTTP.onFillable(HttpConnectionOverHTTP.java:172)
    at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:311)
    at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:105)
    at org.eclipse.jetty.io.ssl.SslConnection$DecryptedEndPoint.onFillable(SslConnection.java:555)
    at org.eclipse.jetty.io.ssl.SslConnection.onFillable(SslConnection.java:410)
    at org.eclipse.jetty.io.ssl.SslConnection$2.succeeded(SslConnection.java:164)
    at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:105)
    at org.eclipse.jetty.io.ChannelEndPoint$1.run(ChannelEndPoint.java:104)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.runTask(EatWhatYouKill.java:338)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:315)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:173)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:131)
    at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:409)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

    Locked monitors: count = 1
      - java.lang.Object@e1a5e10 locked at
          2 java.base@17.0.7/sun.nio.ch.EPollSelectorImpl.wakeup(EPollSelectorImpl.java:251)

"EasyFamilyHttpClient-653" Id=653 in RUNNABLE
    at java.base@17.0.7/sun.nio.ch.EPoll.wait(Native Method)
    at java.base@17.0.7/sun.nio.ch.EPollSelectorImpl.doSelect(EPollSelectorImpl.java:118)
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - locked sun.nio.ch.Util$2@7c0935bc
      - locked sun.nio.ch.EPollSelectorImpl@6c9c40ac
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.select(SelectorImpl.java:146)
    at org.eclipse.jetty.io.ManagedSelector.nioSelect(ManagedSelector.java:183)
    at org.eclipse.jetty.io.ManagedSelector.select(ManagedSelector.java:190)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.select(ManagedSelector.java:606)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.produce(ManagedSelector.java:543)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produceTask(EatWhatYouKill.java:362)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:186)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:173)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:131)
    at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:409)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

    Locked monitors: count = 2
      - sun.nio.ch.Util$2@7c0935bc locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - sun.nio.ch.EPollSelectorImpl@6c9c40ac locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)

"EasyFamilyHttpClient-654" Id=654 in TIMED_WAITING on lock=java.util.concurrent.SynchronousQueue$TransferStack@7f02fb48
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.SynchronousQueue$TransferStack.transfer(SynchronousQueue.java:401)
    at java.base@17.0.7/java.util.concurrent.SynchronousQueue.poll(SynchronousQueue.java:903)
    at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.reservedWait(ReservedThreadExecutor.java:324)
    at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:399)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-655" Id=655 in RUNNABLE
    at java.base@17.0.7/sun.nio.ch.EPoll.wait(Native Method)
    at java.base@17.0.7/sun.nio.ch.EPollSelectorImpl.doSelect(EPollSelectorImpl.java:118)
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - locked sun.nio.ch.Util$2@544253c0
      - locked sun.nio.ch.EPollSelectorImpl@61d32821
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.select(SelectorImpl.java:146)
    at org.eclipse.jetty.io.ManagedSelector.nioSelect(ManagedSelector.java:183)
    at org.eclipse.jetty.io.ManagedSelector.select(ManagedSelector.java:190)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.select(ManagedSelector.java:606)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.produce(ManagedSelector.java:543)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produceTask(EatWhatYouKill.java:362)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:186)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:173)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:131)
    at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:409)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

    Locked monitors: count = 2
      - sun.nio.ch.Util$2@544253c0 locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - sun.nio.ch.EPollSelectorImpl@61d32821 locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)

"EasyFamilyHttpClient-656" Id=656 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@712dd430
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-657" Id=657 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@64ed5989
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-658" Id=658 in TIMED_WAITING on lock=java.util.concurrent.SynchronousQueue$TransferStack@6fc49d97
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.SynchronousQueue$TransferStack.transfer(SynchronousQueue.java:401)
    at java.base@17.0.7/java.util.concurrent.SynchronousQueue.poll(SynchronousQueue.java:903)
    at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.reservedWait(ReservedThreadExecutor.java:324)
    at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:399)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-659" Id=659 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@3372b4a1
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-660" Id=660 in TIMED_WAITING on lock=java.util.concurrent.SynchronousQueue$TransferStack@6fc49d97
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.SynchronousQueue$TransferStack.transfer(SynchronousQueue.java:401)
    at java.base@17.0.7/java.util.concurrent.SynchronousQueue.poll(SynchronousQueue.java:903)
    at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.reservedWait(ReservedThreadExecutor.java:324)
    at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:399)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-661" Id=661 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@7bed5877
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-662" Id=662 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@712dd430
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-663" Id=663 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@712dd430
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"pool-23-thread-1" Id=664 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@46618de2
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1182)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-665" Id=665 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@7bed5877
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-666" Id=666 in RUNNABLE
    at java.base@17.0.7/sun.nio.ch.EPoll.wait(Native Method)
    at java.base@17.0.7/sun.nio.ch.EPollSelectorImpl.doSelect(EPollSelectorImpl.java:118)
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - locked sun.nio.ch.Util$2@75e82d5c
      - locked sun.nio.ch.EPollSelectorImpl@d8281e0
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.select(SelectorImpl.java:146)
    at org.eclipse.jetty.io.ManagedSelector.nioSelect(ManagedSelector.java:183)
    at org.eclipse.jetty.io.ManagedSelector.select(ManagedSelector.java:190)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.select(ManagedSelector.java:606)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.produce(ManagedSelector.java:543)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produceTask(EatWhatYouKill.java:362)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:186)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:173)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:131)
    at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:409)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

    Locked monitors: count = 2
      - sun.nio.ch.Util$2@75e82d5c locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - sun.nio.ch.EPollSelectorImpl@d8281e0 locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)

"EasyFamilyHttpClient-667" Id=667 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@3372b4a1
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-668" Id=668 in TIMED_WAITING on lock=java.util.concurrent.SynchronousQueue$TransferStack@6090fe33
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.SynchronousQueue$TransferStack.transfer(SynchronousQueue.java:401)
    at java.base@17.0.7/java.util.concurrent.SynchronousQueue.poll(SynchronousQueue.java:903)
    at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.reservedWait(ReservedThreadExecutor.java:324)
    at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:399)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-669" Id=669 in RUNNABLE (running in native)
    at java.base@17.0.7/sun.nio.ch.EPoll.wait(Native Method)
    at java.base@17.0.7/sun.nio.ch.EPollSelectorImpl.doSelect(EPollSelectorImpl.java:118)
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - locked sun.nio.ch.Util$2@54a6adb0
      - locked sun.nio.ch.EPollSelectorImpl@4d32ab45
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.select(SelectorImpl.java:146)
    at org.eclipse.jetty.io.ManagedSelector.nioSelect(ManagedSelector.java:183)
    at org.eclipse.jetty.io.ManagedSelector.select(ManagedSelector.java:190)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.select(ManagedSelector.java:606)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.produce(ManagedSelector.java:543)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produceTask(EatWhatYouKill.java:362)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:186)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:173)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:131)
    at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:409)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

    Locked monitors: count = 2
      - sun.nio.ch.Util$2@54a6adb0 locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - sun.nio.ch.EPollSelectorImpl@4d32ab45 locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)

"pool-22-thread-1" Id=670 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@7f6319c
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1182)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-672" Id=672 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@3372b4a1
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"pool-24-thread-1" Id=673 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@15adcb4
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1182)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-scheduler-1" Id=674 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@59ea2759
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1182)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-675" Id=675 in RUNNABLE (running in native)
    at java.base@17.0.7/sun.nio.ch.EPoll.wait(Native Method)
    at java.base@17.0.7/sun.nio.ch.EPollSelectorImpl.doSelect(EPollSelectorImpl.java:118)
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - locked sun.nio.ch.Util$2@6fa68729
      - locked sun.nio.ch.EPollSelectorImpl@e06d574
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.select(SelectorImpl.java:146)
    at org.eclipse.jetty.io.ManagedSelector.nioSelect(ManagedSelector.java:183)
    at org.eclipse.jetty.io.ManagedSelector.select(ManagedSelector.java:190)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.select(ManagedSelector.java:606)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.produce(ManagedSelector.java:543)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produceTask(EatWhatYouKill.java:362)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:186)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:173)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:131)
    at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:409)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

    Locked monitors: count = 2
      - sun.nio.ch.Util$2@6fa68729 locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - sun.nio.ch.EPollSelectorImpl@e06d574 locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)

"EasyFamilyHttpClient-677" Id=677 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@3372b4a1
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at org.eclipse.jetty.util.BlockingArrayQueue.poll(BlockingArrayQueue.java:382)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.idleJobPoll(QueuedThreadPool.java:974)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1018)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"EasyFamilyHttpClient-678" Id=678 in TIMED_WAITING on lock=java.util.concurrent.SynchronousQueue$TransferStack@6090fe33
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.SynchronousQueue$TransferStack.transfer(SynchronousQueue.java:401)
    at java.base@17.0.7/java.util.concurrent.SynchronousQueue.poll(SynchronousQueue.java:903)
    at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.reservedWait(ReservedThreadExecutor.java:324)
    at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:399)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"pool-25-thread-1" Id=679 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@513623f7
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1182)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"OH-org.openhab.binding.km200-1" Id=709 in BLOCKED on lock=org.openhab.binding.km200.internal.KM200Device@3267a068
     owned by OH-org.openhab.binding.km200-2 Id=710
    at org.openhab.binding.km200.internal.handler.KM200GatewayHandler$SendKM200Runnable.run(KM200GatewayHandler.java:612)
    at java.base@17.0.7/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
    at java.base@17.0.7/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:305)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

    Locked synchronizers: count = 1
      - java.util.concurrent.ThreadPoolExecutor$Worker@5701f39d

"OH-org.openhab.binding.km200-2" Id=710 in WAITING on lock=java.util.concurrent.CountDownLatch$Sync@7bd2b5a0
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.park(LockSupport.java:211)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer.acquire(AbstractQueuedSynchronizer.java:715)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer.acquireSharedInterruptibly(AbstractQueuedSynchronizer.java:1047)
    at java.base@17.0.7/java.util.concurrent.CountDownLatch.await(CountDownLatch.java:230)
    at org.eclipse.jetty.client.util.FutureResponseListener.get(FutureResponseListener.java:100)
    at org.eclipse.jetty.client.HttpRequest.send(HttpRequest.java:730)
    at org.openhab.binding.km200.internal.KM200Comm.getDataFromService(KM200Comm.java:71)
    at org.openhab.binding.km200.internal.KM200Device.getServiceNode(KM200Device.java:334)
    at org.openhab.binding.km200.internal.handler.KM200DataHandler.getProvidersState(KM200DataHandler.java:90)
      - locked org.openhab.binding.km200.internal.KM200Device@3267a068
    at org.openhab.binding.km200.internal.handler.KM200GatewayHandler.updateChildren(KM200GatewayHandler.java:460)
      - locked java.util.Collections$SynchronizedMap@1b1b04af
      - locked org.openhab.binding.km200.internal.KM200Device@3267a068
    at org.openhab.binding.km200.internal.handler.KM200GatewayHandler$GetKM200Runnable.run(KM200GatewayHandler.java:513)
      - locked org.openhab.binding.km200.internal.KM200Device@3267a068
    at java.base@17.0.7/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
    at java.base@17.0.7/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:305)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

    Locked synchronizers: count = 1
      - java.util.concurrent.ThreadPoolExecutor$Worker@7823c58d

    Locked monitors: count = 4
      - org.openhab.binding.km200.internal.KM200Device@3267a068 locked at
          9 org.openhab.binding.km200.internal.handler.KM200DataHandler.getProvidersState(KM200DataHandler.java:90)
      - java.util.Collections$SynchronizedMap@1b1b04af locked at
          10 org.openhab.binding.km200.internal.handler.KM200GatewayHandler.updateChildren(KM200GatewayHandler.java:460)
      - org.openhab.binding.km200.internal.KM200Device@3267a068 locked at
          10 org.openhab.binding.km200.internal.handler.KM200GatewayHandler.updateChildren(KM200GatewayHandler.java:460)
      - org.openhab.binding.km200.internal.KM200Device@3267a068 locked at
          11 org.openhab.binding.km200.internal.handler.KM200GatewayHandler$GetKM200Runnable.run(KM200GatewayHandler.java:513)

"Connector-Scheduler-6369d271-1" Id=857 in WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@6270502c
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3463)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3434)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1170)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"pool-2-thread-1" Id=989 in WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@6d1bd8ea
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3463)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3434)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1170)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"OH-rule-676021ab81-1" Id=2420 in WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@2ee6979a
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3463)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3434)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1170)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"OH-rule-bd5c6e4197-1" Id=2697 in WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@40f74853
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3463)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3434)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1170)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"sshd-SshServer[370e5491](port=8101)-nio2-thread-1" Id=4769 in WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@20e69151
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3463)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3434)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623)
    at java.base@17.0.7/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"sshd-SshServer[370e5491](port=8101)-nio2-thread-2" Id=4770 in WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@20e69151
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3463)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3434)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623)
    at java.base@17.0.7/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"qtp726454768-15544" Id=15544 in RUNNABLE (running in native)
    at java.base@17.0.7/sun.nio.ch.EPoll.wait(Native Method)
    at java.base@17.0.7/sun.nio.ch.EPollSelectorImpl.doSelect(EPollSelectorImpl.java:118)
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - locked sun.nio.ch.Util$2@949ce83
      - locked sun.nio.ch.EPollSelectorImpl@7386aa29
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.select(SelectorImpl.java:146)
    at org.eclipse.jetty.io.ManagedSelector.nioSelect(ManagedSelector.java:183)
    at org.eclipse.jetty.io.ManagedSelector.select(ManagedSelector.java:190)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.select(ManagedSelector.java:606)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.produce(ManagedSelector.java:543)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produceTask(EatWhatYouKill.java:362)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:186)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:173)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:131)
    at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:409)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

    Locked monitors: count = 2
      - sun.nio.ch.Util$2@949ce83 locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - sun.nio.ch.EPollSelectorImpl@7386aa29 locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)

"qtp726454768-22005" Id=22005 in RUNNABLE (running in native)
    at java.base@17.0.7/sun.nio.ch.EPoll.wait(Native Method)
    at java.base@17.0.7/sun.nio.ch.EPollSelectorImpl.doSelect(EPollSelectorImpl.java:118)
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - locked sun.nio.ch.Util$2@67fed111
      - locked sun.nio.ch.EPollSelectorImpl@3aa3e478
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.select(SelectorImpl.java:146)
    at org.eclipse.jetty.io.ManagedSelector.nioSelect(ManagedSelector.java:183)
    at org.eclipse.jetty.io.ManagedSelector.select(ManagedSelector.java:190)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.select(ManagedSelector.java:606)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.produce(ManagedSelector.java:543)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produceTask(EatWhatYouKill.java:362)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:186)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:173)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:131)
    at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:409)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

    Locked monitors: count = 2
      - sun.nio.ch.Util$2@67fed111 locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - sun.nio.ch.EPollSelectorImpl@3aa3e478 locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)

"qtp726454768-26815" Id=26815 in RUNNABLE (running in native)
    at java.base@17.0.7/sun.nio.ch.EPoll.wait(Native Method)
    at java.base@17.0.7/sun.nio.ch.EPollSelectorImpl.doSelect(EPollSelectorImpl.java:118)
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - locked sun.nio.ch.Util$2@7dd91879
      - locked sun.nio.ch.EPollSelectorImpl@3b483209
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.select(SelectorImpl.java:146)
    at org.eclipse.jetty.io.ManagedSelector.nioSelect(ManagedSelector.java:183)
    at org.eclipse.jetty.io.ManagedSelector.select(ManagedSelector.java:190)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.select(ManagedSelector.java:606)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.produce(ManagedSelector.java:543)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produceTask(EatWhatYouKill.java:362)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:186)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:173)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:131)
    at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:409)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

    Locked monitors: count = 2
      - sun.nio.ch.Util$2@7dd91879 locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - sun.nio.ch.EPollSelectorImpl@3b483209 locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)

"qtp726454768-27082" Id=27082 in RUNNABLE (running in native)
    at java.base@17.0.7/sun.nio.ch.EPoll.wait(Native Method)
    at java.base@17.0.7/sun.nio.ch.EPollSelectorImpl.doSelect(EPollSelectorImpl.java:118)
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - locked sun.nio.ch.Util$2@2aaf0a28
      - locked sun.nio.ch.EPollSelectorImpl@f8aab21
    at java.base@17.0.7/sun.nio.ch.SelectorImpl.select(SelectorImpl.java:146)
    at org.eclipse.jetty.io.ManagedSelector.nioSelect(ManagedSelector.java:183)
    at org.eclipse.jetty.io.ManagedSelector.select(ManagedSelector.java:190)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.select(ManagedSelector.java:606)
    at org.eclipse.jetty.io.ManagedSelector$SelectorProducer.produce(ManagedSelector.java:543)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produceTask(EatWhatYouKill.java:362)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:186)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:173)
    at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:131)
    at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:409)
    at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
    at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

    Locked monitors: count = 2
      - sun.nio.ch.Util$2@2aaf0a28 locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
      - sun.nio.ch.EPollSelectorImpl@f8aab21 locked at
          2 java.base@17.0.7/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)

"items-queue" Id=27547 in RUNNABLE
    at java.base@17.0.7/java.lang.Object.wait(Native Method)
    at org.openhab.core.common.QueueingThreadPoolExecutor$1.run(QueueingThreadPoolExecutor.java:217)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"OH-discovery-5819" Id=27606 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@3aa57079
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.poll(ScheduledThreadPoolExecutor.java:1218)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.poll(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1061)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"OH-common-7641" Id=27610 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@122b792c
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.poll(ScheduledThreadPoolExecutor.java:1223)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.poll(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1061)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"SocketListener(server-goerentz-internal.local.)" Id=27614 in RUNNABLE
    at java.base@17.0.7/sun.nio.ch.DatagramChannelImpl.receive0(Native Method)
    at java.base@17.0.7/sun.nio.ch.DatagramChannelImpl.receiveIntoNativeBuffer(DatagramChannelImpl.java:750)
    at java.base@17.0.7/sun.nio.ch.DatagramChannelImpl.receive(DatagramChannelImpl.java:728)
    at java.base@17.0.7/sun.nio.ch.DatagramChannelImpl.trustedBlockingReceive(DatagramChannelImpl.java:666)
    at java.base@17.0.7/sun.nio.ch.DatagramChannelImpl.blockingReceive(DatagramChannelImpl.java:635)
    at java.base@17.0.7/sun.nio.ch.DatagramSocketAdaptor.receive(DatagramSocketAdaptor.java:240)
    at java.base@17.0.7/java.net.DatagramSocket.receive(DatagramSocket.java:710)
    at javax.jmdns.impl.SocketListener.run(SocketListener.java:57)

    Locked synchronizers: count = 1
      - java.util.concurrent.locks.ReentrantLock$NonfairSync@6eede0aa

"SocketListener(172-18-0-1.local.)" Id=27617 in RUNNABLE
    at java.base@17.0.7/sun.nio.ch.DatagramChannelImpl.receive0(Native Method)
    at java.base@17.0.7/sun.nio.ch.DatagramChannelImpl.receiveIntoNativeBuffer(DatagramChannelImpl.java:750)
    at java.base@17.0.7/sun.nio.ch.DatagramChannelImpl.receive(DatagramChannelImpl.java:728)
    at java.base@17.0.7/sun.nio.ch.DatagramChannelImpl.trustedBlockingReceive(DatagramChannelImpl.java:666)
    at java.base@17.0.7/sun.nio.ch.DatagramChannelImpl.blockingReceive(DatagramChannelImpl.java:635)
    at java.base@17.0.7/sun.nio.ch.DatagramSocketAdaptor.receive(DatagramSocketAdaptor.java:240)
    at java.base@17.0.7/java.net.DatagramSocket.receive(DatagramSocket.java:710)
    at javax.jmdns.impl.SocketListener.run(SocketListener.java:57)

    Locked synchronizers: count = 1
      - java.util.concurrent.locks.ReentrantLock$NonfairSync@54e3cf6d

"OH-scheduler-2953" Id=27620 in TIMED_WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@1389c12f
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.poll(ScheduledThreadPoolExecutor.java:1218)
    at java.base@17.0.7/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.poll(ScheduledThreadPoolExecutor.java:899)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1061)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"org.apache.karaf.shell.ssh.SshTerminal@584ae156 input pump thread" Id=27621 in WAITING on lock=java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@2806b422
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3463)
    at java.base@17.0.7/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3434)
    at java.base@17.0.7/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623)
    at org.apache.sshd.common.channel.ChannelPipedInputStream.read(ChannelPipedInputStream.java:144)
    at java.base@17.0.7/java.io.InputStream.read(InputStream.java:218)
    at org.jline.terminal.impl.ExternalTerminal.pump(ExternalTerminal.java:141)
    at org.jline.terminal.impl.ExternalTerminal$$Lambda$1392/0x00000007c0e3ea60.run(Unknown Source)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"Karaf ssh console user openhab" Id=27622 in WAITING on lock=org.apache.felix.gogo.runtime.CommandSessionImpl$JobImpl@62d49ee8
    at java.base@17.0.7/java.lang.Object.wait(Native Method)
    at java.base@17.0.7/java.lang.Object.wait(Object.java:338)
    at org.apache.felix.gogo.runtime.CommandSessionImpl$JobImpl.start(CommandSessionImpl.java:802)
    at org.apache.felix.gogo.runtime.Closure.execute(Closure.java:300)
    at org.apache.felix.gogo.runtime.Closure.execute(Closure.java:169)
    at org.apache.felix.gogo.runtime.Closure.execute(Closure.java:149)
    at org.apache.felix.gogo.runtime.CommandSessionImpl.execute(CommandSessionImpl.java:179)
    at org.apache.karaf.shell.impl.console.ConsoleSessionImpl.doExecute(ConsoleSessionImpl.java:467)
    at org.apache.karaf.shell.impl.console.ConsoleSessionImpl.run(ConsoleSessionImpl.java:407)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

"job controller 1" Id=27623 in WAITING on lock=java.util.concurrent.FutureTask@6330a29c
    at java.base@17.0.7/jdk.internal.misc.Unsafe.park(Native Method)
    at java.base@17.0.7/java.util.concurrent.locks.LockSupport.park(LockSupport.java:211)
    at java.base@17.0.7/java.util.concurrent.FutureTask.awaitDone(FutureTask.java:447)
    at java.base@17.0.7/java.util.concurrent.FutureTask.get(FutureTask.java:190)
    at java.base@17.0.7/java.util.concurrent.AbstractExecutorService.invokeAll(AbstractExecutorService.java:252)
    at org.apache.felix.gogo.runtime.CommandSessionImpl$JobImpl.run(CommandSessionImpl.java:827)
    at java.base@17.0.7/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
    at java.base@17.0.7/java.util.concurrent.FutureTask.run(FutureTask.java:264)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

    Locked synchronizers: count = 1
      - java.util.concurrent.ThreadPoolExecutor$Worker@f865f54

"pipe-threads --monitors --locks" Id=27624 in RUNNABLE
    at java.management@17.0.7/sun.management.ThreadImpl.dumpThreads0(Native Method)
    at java.management@17.0.7/sun.management.ThreadImpl.dumpAllThreads(ThreadImpl.java:521)
    at java.management@17.0.7/sun.management.ThreadImpl.dumpAllThreads(ThreadImpl.java:509)
    at org.apache.karaf.shell.commands.impl.ThreadsAction.execute(ThreadsAction.java:78)
    at org.apache.karaf.shell.impl.action.command.ActionCommand.execute(ActionCommand.java:84)
    at org.apache.karaf.shell.impl.console.osgi.secured.SecuredCommand.execute(SecuredCommand.java:68)
    at org.apache.karaf.shell.impl.console.osgi.secured.SecuredCommand.execute(SecuredCommand.java:86)
    at org.apache.felix.gogo.runtime.Closure.executeCmd(Closure.java:599)
    at org.apache.felix.gogo.runtime.Closure.executeStatement(Closure.java:526)
    at org.apache.felix.gogo.runtime.Closure.execute(Closure.java:415)
    at org.apache.felix.gogo.runtime.Pipe.doCall(Pipe.java:416)
    at org.apache.felix.gogo.runtime.Pipe.call(Pipe.java:229)
    at org.apache.felix.gogo.runtime.Pipe.call(Pipe.java:59)
    at java.base@17.0.7/java.util.concurrent.FutureTask.run(FutureTask.java:264)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
    at java.base@17.0.7/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)

    Locked synchronizers: count = 1
      - java.util.concurrent.ThreadPoolExecutor$Worker@4fbf1299

"Common-Cleaner" Id=11 in TIMED_WAITING on lock=java.lang.ref.ReferenceQueue$Lock@5c13385
    at java.base@17.0.7/java.lang.Object.wait(Native Method)
    at java.base@17.0.7/java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:155)
    at java.base@17.0.7/jdk.internal.ref.CleanerImpl.run(CleanerImpl.java:140)
    at java.base@17.0.7/java.lang.Thread.run(Thread.java:833)
    at java.base@17.0.7/jdk.internal.misc.InnocuousThread.run(InnocuousThread.java:162)
</details>

Reply:
4.0.0 instance is running on a Windows machine while 3.4.3 instance is running on a Linux server.

Reply:
@lolodomo - since you were already doing a review of this PR, perhaps I could ask if you have any ideas what could be the cause of the problem @marcelGoerentz described in this comment? I checked the output above, and don't see anything unusual regarding blocked threads in the 3.4.x version, and liquidcheck is not mentioned at all.